{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "These commands allow me to install BWA, SAMtools, and BCFtools. BWA allows me to align short DNA reads to a reference genome, using BWT. SAMtools allows me to read, write, sort, and manipulate sequence alignment data in SAM/BAM format. BCFtools allows me to call variants and manipulate VCF files."
      ],
      "metadata": {
        "id": "id_gai0wpgOW"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!apt install bwa\n",
        "!apt install samtools\n",
        "!apt install bcftools"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-4JabQzCkXo3",
        "outputId": "5912ce85-5322-4b9e-db9b-261817f5c489"
      },
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  liblmdb0 ncbi-data\n",
            "The following NEW packages will be installed:\n",
            "  liblmdb0 ncbi-blast+ ncbi-data\n",
            "0 upgraded, 3 newly installed, 0 to remove and 49 not upgraded.\n",
            "Need to get 15.9 MB of archives.\n",
            "After this operation, 71.9 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy/main amd64 liblmdb0 amd64 0.9.24-1build2 [47.6 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy/universe amd64 ncbi-data all 6.1.20170106+dfsg1-9 [3,519 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu jammy/universe amd64 ncbi-blast+ amd64 2.12.0+ds-3build1 [12.3 MB]\n",
            "Fetched 15.9 MB in 0s (52.9 MB/s)\n",
            "Selecting previously unselected package liblmdb0:amd64.\n",
            "(Reading database ... 123632 files and directories currently installed.)\n",
            "Preparing to unpack .../liblmdb0_0.9.24-1build2_amd64.deb ...\n",
            "Unpacking liblmdb0:amd64 (0.9.24-1build2) ...\n",
            "Selecting previously unselected package ncbi-data.\n",
            "Preparing to unpack .../ncbi-data_6.1.20170106+dfsg1-9_all.deb ...\n",
            "Unpacking ncbi-data (6.1.20170106+dfsg1-9) ...\n",
            "Selecting previously unselected package ncbi-blast+.\n",
            "Preparing to unpack .../ncbi-blast+_2.12.0+ds-3build1_amd64.deb ...\n",
            "Unpacking ncbi-blast+ (2.12.0+ds-3build1) ...\n",
            "Setting up liblmdb0:amd64 (0.9.24-1build2) ...\n",
            "Setting up ncbi-data (6.1.20170106+dfsg1-9) ...\n",
            "Setting up ncbi-blast+ (2.12.0+ds-3build1) ...\n",
            "Processing triggers for libc-bin (2.35-0ubuntu3.4) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtcm_debug.so.1 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_loader.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_opencl.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libhwloc.so.15 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libumf.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtcm.so.1 is not a symbolic link\n",
            "\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "Processing triggers for hicolor-icon-theme (0.17-2) ...\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "Suggested packages:\n",
            "  samtools\n",
            "The following NEW packages will be installed:\n",
            "  bwa\n",
            "0 upgraded, 1 newly installed, 0 to remove and 49 not upgraded.\n",
            "Need to get 195 kB of archives.\n",
            "After this operation, 466 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy/universe amd64 bwa amd64 0.7.17-6 [195 kB]\n",
            "Fetched 195 kB in 0s (2,447 kB/s)\n",
            "Selecting previously unselected package bwa.\n",
            "(Reading database ... 123873 files and directories currently installed.)\n",
            "Preparing to unpack .../bwa_0.7.17-6_amd64.deb ...\n",
            "Unpacking bwa (0.7.17-6) ...\n",
            "Setting up bwa (0.7.17-6) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  libhts3 libhtscodecs2\n",
            "Suggested packages:\n",
            "  cwltool\n",
            "The following NEW packages will be installed:\n",
            "  libhts3 libhtscodecs2 samtools\n",
            "0 upgraded, 3 newly installed, 0 to remove and 49 not upgraded.\n",
            "Need to get 963 kB of archives.\n",
            "After this operation, 2,270 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhtscodecs2 amd64 1.1.1-3 [53.2 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhts3 amd64 1.13+ds-2build1 [390 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu jammy/universe amd64 samtools amd64 1.13-4 [520 kB]\n",
            "Fetched 963 kB in 0s (7,869 kB/s)\n",
            "Selecting previously unselected package libhtscodecs2:amd64.\n",
            "(Reading database ... 123890 files and directories currently installed.)\n",
            "Preparing to unpack .../libhtscodecs2_1.1.1-3_amd64.deb ...\n",
            "Unpacking libhtscodecs2:amd64 (1.1.1-3) ...\n",
            "Selecting previously unselected package libhts3:amd64.\n",
            "Preparing to unpack .../libhts3_1.13+ds-2build1_amd64.deb ...\n",
            "Unpacking libhts3:amd64 (1.13+ds-2build1) ...\n",
            "Selecting previously unselected package samtools.\n",
            "Preparing to unpack .../samtools_1.13-4_amd64.deb ...\n",
            "Unpacking samtools (1.13-4) ...\n",
            "Setting up libhtscodecs2:amd64 (1.1.1-3) ...\n",
            "Setting up libhts3:amd64 (1.13+ds-2build1) ...\n",
            "Setting up samtools (1.13-4) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "Processing triggers for libc-bin (2.35-0ubuntu3.4) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtcm_debug.so.1 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_loader.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_opencl.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libhwloc.so.15 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libumf.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtcm.so.1 is not a symbolic link\n",
            "\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "Suggested packages:\n",
            "  python3-numpy python3-matplotlib texlive-latex-recommended\n",
            "The following NEW packages will be installed:\n",
            "  bcftools\n",
            "0 upgraded, 1 newly installed, 0 to remove and 49 not upgraded.\n",
            "Need to get 697 kB of archives.\n",
            "After this operation, 2,360 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy/universe amd64 bcftools amd64 1.13-1 [697 kB]\n",
            "Fetched 697 kB in 0s (6,688 kB/s)\n",
            "Selecting previously unselected package bcftools.\n",
            "(Reading database ... 123989 files and directories currently installed.)\n",
            "Preparing to unpack .../bcftools_1.13-1_amd64.deb ...\n",
            "Unpacking bcftools (1.13-1) ...\n",
            "Setting up bcftools (1.13-1) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "!gdown allows me to download files. CRISPR.R1.fastq, CRISPR.R2.fastq, and MM10.tar.gz were downloaded respectively."
      ],
      "metadata": {
        "id": "uZBosNfppq-h"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ElE9ARnjjmLo",
        "outputId": "2a326b30-bfa9-429e-ba53-1fb1192f8718"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloading...\n",
            "From (original): https://drive.google.com/uc?id=1-96T1PZKA_FQeD_ZK5z9USaHLP3jVdRO\n",
            "From (redirected): https://drive.google.com/uc?id=1-96T1PZKA_FQeD_ZK5z9USaHLP3jVdRO&confirm=t&uuid=8837f615-0a0a-432c-9b4c-4075db6651fa\n",
            "To: /content/CRISPR.R1.fastq\n",
            "100% 2.67G/2.67G [00:43<00:00, 61.4MB/s]\n",
            "Downloading...\n",
            "From (original): https://drive.google.com/uc?id=1-BXGr3XVGtd9Tx6PCHSp4hepK41GGCTS\n",
            "From (redirected): https://drive.google.com/uc?id=1-BXGr3XVGtd9Tx6PCHSp4hepK41GGCTS&confirm=t&uuid=f1a09737-a719-484c-87ef-2e581e0c48a7\n",
            "To: /content/CRISPR.R2.fastq\n",
            "100% 2.67G/2.67G [00:27<00:00, 96.9MB/s]\n",
            "Downloading...\n",
            "From (original): https://drive.google.com/uc?id=1a8CP4P5zkzIBiw1EleqJiSwDW0VZcAar\n",
            "From (redirected): https://drive.google.com/uc?id=1a8CP4P5zkzIBiw1EleqJiSwDW0VZcAar&confirm=t&uuid=e5981858-a3d6-4448-852d-961b2264e0f0\n",
            "To: /content/MM10.tar.gz\n",
            "100% 3.91G/3.91G [01:18<00:00, 50.1MB/s]\n"
          ]
        }
      ],
      "source": [
        "!gdown 1-96T1PZKA_FQeD_ZK5z9USaHLP3jVdRO\n",
        "!gdown 1-BXGr3XVGtd9Tx6PCHSp4hepK41GGCTS\n",
        "!gdown 1a8CP4P5zkzIBiw1EleqJiSwDW0VZcAar"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "!tar allows me to extract data from MM10.tar.gz. -x tells tar to extract files from an archive. -z specifies that the archive is compressed with gzip. -f states that the next argument is the name of the archive file that will be extracted."
      ],
      "metadata": {
        "id": "BwsjSpAlpwQC"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!tar -xzf MM10.tar.gz"
      ],
      "metadata": {
        "id": "lyYCqWNeqveC"
      },
      "execution_count": 4,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "!bwa index makes an index with the reference genome, which is Mouse.fasta (the path is MM10/Mouse.fasta). -p MM10 specifies that the generated index files should be prefixed with MM10.\n",
        "\n"
      ],
      "metadata": {
        "id": "-Op3a3vQmlKg"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!bwa index -p MM10 MM10/Mouse.fasta"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "isPjDnR-jkBi",
        "outputId": "ccc8c3da-deb3-489f-d57e-7603a042ab68"
      },
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[bwa_index] Pack FASTA... 29.76 sec\n",
            "[bwa_index] Construct BWT for the packed sequence...\n",
            "[BWTIncCreate] textLength=5451075338, availableWord=395557796\n",
            "[BWTIncConstructFromPacked] 10 iterations done. 99999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 20 iterations done. 199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 30 iterations done. 299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 40 iterations done. 399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 50 iterations done. 499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 60 iterations done. 599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 70 iterations done. 699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 80 iterations done. 799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 90 iterations done. 899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 100 iterations done. 999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 110 iterations done. 1099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 120 iterations done. 1199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 130 iterations done. 1299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 140 iterations done. 1399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 150 iterations done. 1499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 160 iterations done. 1599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 170 iterations done. 1699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 180 iterations done. 1799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 190 iterations done. 1899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 200 iterations done. 1999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 210 iterations done. 2099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 220 iterations done. 2199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 230 iterations done. 2299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 240 iterations done. 2399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 250 iterations done. 2499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 260 iterations done. 2599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 270 iterations done. 2699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 280 iterations done. 2799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 290 iterations done. 2899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 300 iterations done. 2999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 310 iterations done. 3099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 320 iterations done. 3199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 330 iterations done. 3299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 340 iterations done. 3399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 350 iterations done. 3499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 360 iterations done. 3599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 370 iterations done. 3699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 380 iterations done. 3799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 390 iterations done. 3899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 400 iterations done. 3999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 410 iterations done. 4099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 420 iterations done. 4199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 430 iterations done. 4299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 440 iterations done. 4399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 450 iterations done. 4499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 460 iterations done. 4599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 470 iterations done. 4699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 480 iterations done. 4799664682 characters processed.\n",
            "[BWTIncConstructFromPacked] 490 iterations done. 4891322586 characters processed.\n",
            "[BWTIncConstructFromPacked] 500 iterations done. 4972784378 characters processed.\n",
            "[BWTIncConstructFromPacked] 510 iterations done. 5045183802 characters processed.\n",
            "[BWTIncConstructFromPacked] 520 iterations done. 5109528586 characters processed.\n",
            "[BWTIncConstructFromPacked] 530 iterations done. 5166714378 characters processed.\n",
            "[BWTIncConstructFromPacked] 540 iterations done. 5217537242 characters processed.\n",
            "[BWTIncConstructFromPacked] 550 iterations done. 5262704698 characters processed.\n",
            "[BWTIncConstructFromPacked] 560 iterations done. 5302845626 characters processed.\n",
            "[BWTIncConstructFromPacked] 570 iterations done. 5338518970 characters processed.\n",
            "[BWTIncConstructFromPacked] 580 iterations done. 5370221514 characters processed.\n",
            "[BWTIncConstructFromPacked] 590 iterations done. 5398394762 characters processed.\n",
            "[BWTIncConstructFromPacked] 600 iterations done. 5423431210 characters processed.\n",
            "[BWTIncConstructFromPacked] 610 iterations done. 5445679594 characters processed.\n",
            "[bwt_gen] Finished constructing BWT in 613 iterations.\n",
            "[bwa_index] 4121.43 seconds elapse.\n",
            "[bwa_index] Update BWT... 21.39 sec\n",
            "[bwa_index] Pack forward-only FASTA... 14.98 sec\n",
            "[bwa_index] Construct SA from BWT and Occ... 2140.95 sec\n",
            "[main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa index -p MM10 MM10/Mouse.fasta\n",
            "[main] Real time: 6593.334 sec; CPU: 6328.517 sec\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "!bwa mem aligns both CRISPR FASTQ files to the reference genome which is indexed with the prefix MM10.  \"> alignedreads.sam\" allows the alignments to be saved in the SAM format to \"alignedreads.sam."
      ],
      "metadata": {
        "id": "FDWyDeC-rZkG"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!bwa mem MM10 CRISPR.R1.fastq CRISPR.R2.fastq > alignedreads.sam"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "m8pMgxdceUZc",
        "outputId": "0b55e7e8-fed3-41f2-96e6-d10c01fb9e21"
      },
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46782, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 505)\n",
            "[M::mem_pestat] mean and std.dev: (263.49, 57.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 604)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.769 CPU sec, 20.916 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47080, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.51, 57.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.031 CPU sec, 21.092 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47089, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.53, 56.71)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.363 CPU sec, 20.454 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44970, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (264.06, 56.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.565 CPU sec, 20.624 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47154, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.39, 57.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.880 CPU sec, 19.976 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (3, 47303, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.36, 56.91)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.090 CPU sec, 20.181 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44526, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.20, 57.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.944 CPU sec, 21.085 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46776, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.69, 56.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.231 CPU sec, 24.052 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46512, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 505)\n",
            "[M::mem_pestat] mean and std.dev: (263.19, 57.38)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 604)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.159 CPU sec, 23.390 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46460, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.32, 57.56)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.275 CPU sec, 21.448 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47045, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.02, 57.16)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.344 CPU sec, 20.493 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46776, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.40, 57.38)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.917 CPU sec, 21.078 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47253, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.85, 56.69)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.181 CPU sec, 20.316 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45312, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.31, 57.28)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.178 CPU sec, 20.216 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46917, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.05, 57.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.854 CPU sec, 19.951 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46955, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.96, 57.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.748 CPU sec, 19.801 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46844, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.68, 57.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.510 CPU sec, 20.612 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46811, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.44, 57.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 18.057 CPU sec, 18.058 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47067, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.53, 57.55)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.100 CPU sec, 20.387 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47697, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.29, 57.05)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 25.919 CPU sec, 27.032 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (4, 47725, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.69, 56.98)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.466 CPU sec, 20.570 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47394, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.51, 57.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.813 CPU sec, 25.620 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 44901, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.23, 57.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.420 CPU sec, 21.563 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46399, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.12, 57.28)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.915 CPU sec, 21.031 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46947, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.21, 56.74)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.052 CPU sec, 20.209 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46660, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.40, 57.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.637 CPU sec, 19.712 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47022, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.73, 56.98)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.105 CPU sec, 20.264 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47467, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.67, 56.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.055 CPU sec, 20.181 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44268, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.32, 57.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.394 CPU sec, 20.578 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46414, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 508)\n",
            "[M::mem_pestat] mean and std.dev: (263.65, 57.67)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 608)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.437 CPU sec, 20.615 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 41605, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (263.91, 57.00)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.334 CPU sec, 23.641 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46652, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.19, 56.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.473 CPU sec, 21.662 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46698, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.05, 56.93)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.192 CPU sec, 21.342 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47138, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.62, 56.88)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.525 CPU sec, 20.641 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47388, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.09, 57.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.365 CPU sec, 21.574 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45945, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.25, 57.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.277 CPU sec, 21.452 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47579, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 508)\n",
            "[M::mem_pestat] mean and std.dev: (263.84, 57.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 608)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.055 CPU sec, 21.301 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 42483, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.92, 57.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.220 CPU sec, 21.380 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46299, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.70, 57.33)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.455 CPU sec, 20.634 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47073, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.42, 57.43)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.395 CPU sec, 21.502 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47434, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.50, 57.11)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.177 CPU sec, 21.292 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47570, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.29, 56.88)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.160 CPU sec, 19.167 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47122, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.33, 56.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.873 CPU sec, 20.069 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46209, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.88, 56.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.536 CPU sec, 20.778 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (3, 47128, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.31, 57.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.930 CPU sec, 22.059 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47620, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.95, 57.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.142 CPU sec, 21.325 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 42295, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.97, 57.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.802 CPU sec, 22.045 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44128, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.78, 57.45)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.063 CPU sec, 23.266 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47232, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.21, 56.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.830 CPU sec, 20.983 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47272, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.48, 56.83)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.440 CPU sec, 20.579 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47410, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (264.09, 56.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.792 CPU sec, 26.061 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 46449, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.04, 57.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.001 CPU sec, 20.154 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 44326, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 508)\n",
            "[M::mem_pestat] mean and std.dev: (263.88, 57.72)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 608)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.937 CPU sec, 21.144 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46005, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 310)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 512)\n",
            "[M::mem_pestat] mean and std.dev: (264.28, 58.58)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 613)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.296 CPU sec, 21.514 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 39143, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.08, 57.74)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.990 CPU sec, 23.125 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45624, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.51, 57.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.834 CPU sec, 20.013 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 42308, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.42, 57.25)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.977 CPU sec, 24.124 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 45151, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.47, 56.77)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.860 CPU sec, 21.063 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 47042, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 258, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.86, 57.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.535 CPU sec, 19.690 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47642, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.02, 56.95)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.001 CPU sec, 20.169 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47332, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.58, 57.44)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.086 CPU sec, 22.331 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46869, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.93, 57.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.440 CPU sec, 20.571 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47203, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.23, 57.11)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.930 CPU sec, 21.089 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 42633, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.90, 57.01)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.067 CPU sec, 22.312 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44822, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.41, 57.49)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.106 CPU sec, 21.245 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45711, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 511)\n",
            "[M::mem_pestat] mean and std.dev: (263.92, 58.48)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 612)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.004 CPU sec, 22.203 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45890, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (207, 254, 306)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 504)\n",
            "[M::mem_pestat] mean and std.dev: (262.79, 58.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 603)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.674 CPU sec, 21.805 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 44188, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 505)\n",
            "[M::mem_pestat] mean and std.dev: (263.19, 57.98)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 604)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.597 CPU sec, 20.732 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46399, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.73, 57.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.747 CPU sec, 20.892 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47816, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.53, 56.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.632 CPU sec, 20.800 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47377, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.08, 57.02)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.002 CPU sec, 20.124 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 45631, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.32, 57.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.206 CPU sec, 21.399 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46382, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (264.02, 56.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.405 CPU sec, 20.548 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 43643, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.42, 57.26)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.275 CPU sec, 22.422 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 45932, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.61, 57.36)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.623 CPU sec, 21.785 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46956, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.96, 56.91)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.914 CPU sec, 20.013 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47421, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.60, 57.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.277 CPU sec, 20.462 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46647, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.21, 57.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.253 CPU sec, 20.474 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 41089, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.19, 57.25)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 25.127 CPU sec, 26.598 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 47083, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.42, 57.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.521 CPU sec, 20.744 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 34042, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 254, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 505)\n",
            "[M::mem_pestat] mean and std.dev: (262.96, 57.40)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 604)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 26.874 CPU sec, 27.100 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (3, 33934, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 310)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 514)\n",
            "[M::mem_pestat] mean and std.dev: (263.91, 58.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 616)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 26.681 CPU sec, 26.920 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 37493, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.92, 57.64)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.853 CPU sec, 24.055 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 46357, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.15, 57.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.469 CPU sec, 20.638 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 42733, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.19, 56.79)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 26.057 CPU sec, 26.354 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47198, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.08, 57.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.268 CPU sec, 20.420 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46909, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.25, 57.34)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.938 CPU sec, 21.183 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45421, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.87, 57.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.232 CPU sec, 20.421 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45557, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.80, 57.21)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.066 CPU sec, 20.156 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46280, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 306)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 502)\n",
            "[M::mem_pestat] mean and std.dev: (262.90, 57.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 600)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.787 CPU sec, 20.989 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47156, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.72, 56.39)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.146 CPU sec, 19.271 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46389, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.71, 56.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.203 CPU sec, 19.337 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47711, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (264.01, 56.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.399 CPU sec, 19.519 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 41450, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.59, 56.82)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.964 CPU sec, 22.060 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47227, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.07, 57.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.484 CPU sec, 19.511 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47236, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.53, 56.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.817 CPU sec, 20.908 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46680, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.31, 57.29)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.744 CPU sec, 21.956 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 45189, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.00, 57.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.096 CPU sec, 21.249 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46967, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.76, 57.10)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.907 CPU sec, 20.050 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47069, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.27, 57.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.826 CPU sec, 20.955 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45777, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.79, 57.44)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.548 CPU sec, 20.653 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47049, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.15, 57.18)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.499 CPU sec, 20.655 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45551, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.18, 57.30)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.727 CPU sec, 21.978 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46931, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.91, 57.12)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.292 CPU sec, 21.475 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47793, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (264.15, 56.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.790 CPU sec, 19.924 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46377, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.25, 57.12)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.646 CPU sec, 20.908 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46830, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.44, 57.34)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.720 CPU sec, 25.446 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46476, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.04, 57.78)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.934 CPU sec, 21.052 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46984, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.88, 57.69)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.863 CPU sec, 21.027 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44475, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.96, 56.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.545 CPU sec, 20.681 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47237, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.12, 57.31)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.381 CPU sec, 20.492 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46109, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.88, 57.21)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.517 CPU sec, 20.693 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 45581, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 505)\n",
            "[M::mem_pestat] mean and std.dev: (263.13, 57.73)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 604)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.155 CPU sec, 21.356 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46570, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.87, 57.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.618 CPU sec, 20.747 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (3, 46108, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (264.04, 56.72)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.414 CPU sec, 20.666 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47428, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.97, 56.96)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.507 CPU sec, 20.686 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46783, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.13, 57.58)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.927 CPU sec, 21.090 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46822, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.20, 57.66)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.093 CPU sec, 21.253 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47327, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.68, 57.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.570 CPU sec, 21.734 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44605, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.64, 57.25)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.151 CPU sec, 20.233 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46936, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.90, 57.12)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.273 CPU sec, 21.473 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47295, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.28, 56.98)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.177 CPU sec, 21.422 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45927, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.13, 57.16)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.376 CPU sec, 20.485 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45677, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.10, 57.32)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.287 CPU sec, 20.374 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47468, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.87, 56.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.751 CPU sec, 19.880 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46902, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.26, 57.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.537 CPU sec, 19.633 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47666, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (211, 258, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (15, 505)\n",
            "[M::mem_pestat] mean and std.dev: (265.18, 57.10)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 603)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.895 CPU sec, 20.132 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 45132, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.66, 57.33)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.104 CPU sec, 20.237 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46320, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.88, 57.33)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.373 CPU sec, 20.575 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46407, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.14, 57.40)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.862 CPU sec, 19.996 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46252, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.59, 58.11)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.766 CPU sec, 20.909 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46605, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.85, 57.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.224 CPU sec, 21.521 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45855, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.32, 58.00)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.797 CPU sec, 20.920 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46565, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.82, 57.56)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.406 CPU sec, 20.509 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46112, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.66, 57.26)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.866 CPU sec, 20.021 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 34558, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.68, 57.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 32.537 CPU sec, 34.027 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 40731, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.21, 56.52)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 24.312 CPU sec, 24.480 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47728, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (264.03, 56.65)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.974 CPU sec, 20.149 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46472, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (211, 258, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (15, 505)\n",
            "[M::mem_pestat] mean and std.dev: (265.00, 57.21)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 603)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.078 CPU sec, 20.229 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47237, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.01, 57.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.034 CPU sec, 21.210 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46639, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.20, 57.56)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.234 CPU sec, 20.425 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 45198, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.19, 57.41)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.777 CPU sec, 20.907 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 45683, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.98, 57.32)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.139 CPU sec, 20.282 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 43815, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.10, 56.83)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.046 CPU sec, 20.228 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 45857, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.86, 56.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.390 CPU sec, 20.622 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 43209, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.84, 56.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.629 CPU sec, 20.751 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46212, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.60, 56.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.866 CPU sec, 21.032 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47187, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.73, 56.91)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.002 CPU sec, 21.200 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 43693, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.65, 57.09)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.772 CPU sec, 20.954 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 37905, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.53, 56.82)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 24.640 CPU sec, 24.869 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47509, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (211, 257, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (19, 499)\n",
            "[M::mem_pestat] mean and std.dev: (264.47, 56.19)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 595)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.774 CPU sec, 19.977 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47443, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 255, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.01, 56.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.247 CPU sec, 20.380 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46660, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.51, 56.87)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.510 CPU sec, 20.667 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47443, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.15, 57.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.592 CPU sec, 20.785 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 24541, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 306)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (15, 500)\n",
            "[M::mem_pestat] mean and std.dev: (263.42, 56.64)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 597)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 38.287 CPU sec, 38.579 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 45924, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.46, 57.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.258 CPU sec, 20.375 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45815, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.60, 56.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.975 CPU sec, 20.125 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 34880, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.55, 56.76)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 27.173 CPU sec, 27.437 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 41700, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.45, 57.34)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.138 CPU sec, 22.307 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45711, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.79, 56.78)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.206 CPU sec, 20.405 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46212, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.23, 56.93)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.512 CPU sec, 20.770 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47860, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (211, 258, 310)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 508)\n",
            "[M::mem_pestat] mean and std.dev: (265.45, 57.53)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 607)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.416 CPU sec, 21.598 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47646, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.97, 57.16)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.259 CPU sec, 20.390 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47674, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.71, 57.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 24.335 CPU sec, 26.476 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46659, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.58, 57.01)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.331 CPU sec, 20.532 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47624, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.66, 57.32)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.012 CPU sec, 21.178 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47218, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.00, 56.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.067 CPU sec, 20.152 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47627, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.50, 57.38)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.819 CPU sec, 21.015 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47220, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.62, 56.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.982 CPU sec, 20.154 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47119, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.81, 56.48)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.654 CPU sec, 19.803 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44731, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 310)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 510)\n",
            "[M::mem_pestat] mean and std.dev: (264.93, 57.46)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 610)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.664 CPU sec, 19.740 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47258, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.25, 56.90)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.403 CPU sec, 19.574 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 45267, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.53, 57.49)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.299 CPU sec, 20.458 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46598, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 505)\n",
            "[M::mem_pestat] mean and std.dev: (263.34, 57.48)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 604)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.405 CPU sec, 20.530 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46446, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.83, 56.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.058 CPU sec, 20.196 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46318, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.61, 57.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.191 CPU sec, 20.390 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47554, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.40, 56.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.937 CPU sec, 20.040 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 47374, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.34, 56.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.746 CPU sec, 20.901 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47951, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (211, 258, 310)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 508)\n",
            "[M::mem_pestat] mean and std.dev: (265.55, 57.23)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 607)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.837 CPU sec, 21.046 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47724, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.00, 56.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.685 CPU sec, 20.838 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46408, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 306)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 502)\n",
            "[M::mem_pestat] mean and std.dev: (263.23, 56.98)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 600)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.573 CPU sec, 20.683 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 39829, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.87, 56.43)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.343 CPU sec, 20.547 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 41544, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.32, 56.86)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.488 CPU sec, 21.661 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 43261, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.39, 57.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.064 CPU sec, 21.234 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46179, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.21, 57.51)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.160 CPU sec, 21.310 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46776, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.32, 57.10)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.421 CPU sec, 21.594 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 47183, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (264.15, 57.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.469 CPU sec, 20.695 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47030, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.95, 56.98)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.629 CPU sec, 20.778 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46340, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.19, 57.24)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.458 CPU sec, 20.574 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46639, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.05, 57.46)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.602 CPU sec, 20.703 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47498, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (264.04, 56.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.287 CPU sec, 20.493 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 46778, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.64, 57.24)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.738 CPU sec, 20.912 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47557, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.68, 56.82)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 24.129 CPU sec, 25.692 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47246, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.70, 57.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.675 CPU sec, 20.866 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47254, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (263.88, 56.83)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.993 CPU sec, 20.144 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47320, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (16, 501)\n",
            "[M::mem_pestat] mean and std.dev: (264.10, 56.53)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 598)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 19.056 CPU sec, 19.173 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44322, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.10, 57.28)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.686 CPU sec, 20.882 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 47231, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (13, 503)\n",
            "[M::mem_pestat] mean and std.dev: (263.41, 56.81)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 601)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.534 CPU sec, 20.763 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 46445, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 257, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.40, 56.95)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.351 CPU sec, 21.700 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 46041, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 257, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.40, 57.65)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.689 CPU sec, 20.836 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 47528, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 306)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (15, 500)\n",
            "[M::mem_pestat] mean and std.dev: (263.42, 56.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 597)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 20.511 CPU sec, 20.701 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 40935, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 256, 307)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 505)\n",
            "[M::mem_pestat] mean and std.dev: (263.40, 57.30)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 604)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 24.530 CPU sec, 24.860 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 31129, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 254, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 508)\n",
            "[M::mem_pestat] mean and std.dev: (263.27, 58.10)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 608)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 32.611 CPU sec, 33.034 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 43372, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 507)\n",
            "[M::mem_pestat] mean and std.dev: (264.59, 57.76)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 606)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.881 CPU sec, 23.121 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 44289, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (210, 256, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (14, 504)\n",
            "[M::mem_pestat] mean and std.dev: (264.04, 57.14)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 602)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.310 CPU sec, 21.427 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 43422, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 511)\n",
            "[M::mem_pestat] mean and std.dev: (263.91, 57.67)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 612)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.515 CPU sec, 22.685 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 44069, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.00, 58.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 21.670 CPU sec, 21.850 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 40367, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 255, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 511)\n",
            "[M::mem_pestat] mean and std.dev: (264.04, 57.91)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 612)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 29.689 CPU sec, 30.032 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 44607, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.13, 57.53)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 22.364 CPU sec, 22.728 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 37344, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 256, 309)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 509)\n",
            "[M::mem_pestat] mean and std.dev: (264.29, 57.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 609)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 26.131 CPU sec, 26.366 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 28339, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (209, 255, 308)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 506)\n",
            "[M::mem_pestat] mean and std.dev: (263.72, 57.56)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 605)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 61.751 CPU sec, 62.499 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 74, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (200, 246, 298)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 494)\n",
            "[M::mem_pestat] mean and std.dev: (256.43, 57.41)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 592)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 130.551 CPU sec, 132.707 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 9, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation FR as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 65.717 CPU sec, 70.439 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 0, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation FR as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 57.095 CPU sec, 57.866 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 1, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation FR as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 100000 reads in 63.735 CPU sec, 64.439 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 6807, 264, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (204, 246, 292)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (28, 468)\n",
            "[M::mem_pestat] mean and std.dev: (248.27, 48.76)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 556)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (715, 4998, 6889)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 19237)\n",
            "[M::mem_pestat] mean and std.dev: (3792.68, 2843.45)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 25411)\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_process_seqs] Processed 100000 reads in 118.231 CPU sec, 119.640 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (256, 26395, 716, 134)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (5934, 7042, 9226)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15810)\n",
            "[M::mem_pestat] mean and std.dev: (7020.39, 1999.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19102)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (202, 244, 286)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (34, 454)\n",
            "[M::mem_pestat] mean and std.dev: (248.88, 47.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 538)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (842, 4528, 8564)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 24008)\n",
            "[M::mem_pestat] mean and std.dev: (4415.10, 3448.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 31730)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (5829, 8417, 8805)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 14757)\n",
            "[M::mem_pestat] mean and std.dev: (6357.01, 3083.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 18693)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 10.249 CPU sec, 10.172 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (25, 31532, 1259, 13)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (5716, 6952, 7065)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (3018, 9763)\n",
            "[M::mem_pestat] mean and std.dev: (6445.36, 668.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1669, 11112)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 258, 331)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 577)\n",
            "[M::mem_pestat] mean and std.dev: (251.34, 59.58)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 700)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (711, 1930, 3661)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 9561)\n",
            "[M::mem_pestat] mean and std.dev: (2236.95, 1905.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 12511)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (9310, 9325, 9378)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9174, 9514)\n",
            "[M::mem_pestat] mean and std.dev: (9336.54, 39.39)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (9106, 9582)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 13.666 CPU sec, 13.834 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (416, 29466, 1211, 129)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (6205, 8018, 8449)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1717, 12937)\n",
            "[M::mem_pestat] mean and std.dev: (7478.57, 1780.44)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 15181)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (203, 246, 291)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (27, 467)\n",
            "[M::mem_pestat] mean and std.dev: (248.93, 48.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 555)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (685, 828, 6267)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17431)\n",
            "[M::mem_pestat] mean and std.dev: (3362.08, 3280.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 23013)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (8476, 9091, 9305)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6818, 10963)\n",
            "[M::mem_pestat] mean and std.dev: (8809.49, 794.53)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (5631, 11988)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 12.635 CPU sec, 12.654 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (204, 27844, 834, 41)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (3703, 6321, 7565)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15289)\n",
            "[M::mem_pestat] mean and std.dev: (5931.38, 2201.88)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19151)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (203, 245, 289)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (31, 461)\n",
            "[M::mem_pestat] mean and std.dev: (248.36, 48.48)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 547)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (708, 821, 2016)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 4632)\n",
            "[M::mem_pestat] mean and std.dev: (1075.68, 861.36)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 5940)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (8817, 9724, 9941)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6569, 12189)\n",
            "[M::mem_pestat] mean and std.dev: (9451.56, 524.86)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (5445, 13313)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 14.801 CPU sec, 14.923 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (25, 22913, 1325, 48)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (6457, 6516, 6581)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6209, 6829)\n",
            "[M::mem_pestat] mean and std.dev: (6493.86, 56.82)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (6085, 6953)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (204, 248, 297)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (18, 483)\n",
            "[M::mem_pestat] mean and std.dev: (248.39, 49.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 576)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (696, 2437, 4273)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 11427)\n",
            "[M::mem_pestat] mean and std.dev: (2830.31, 2476.00)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 15004)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (6239, 7792, 7872)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (2973, 11138)\n",
            "[M::mem_pestat] mean and std.dev: (7287.21, 780.72)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1340, 12771)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 79.452 CPU sec, 80.366 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (139, 23690, 1754, 24)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2120, 6269, 7146)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17198)\n",
            "[M::mem_pestat] mean and std.dev: (5372.67, 2558.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22224)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (202, 246, 294)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (18, 478)\n",
            "[M::mem_pestat] mean and std.dev: (247.31, 49.54)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 570)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (749, 2816, 4991)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 13475)\n",
            "[M::mem_pestat] mean and std.dev: (3189.96, 2681.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 17717)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2351, 2387, 2432)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (2189, 2594)\n",
            "[M::mem_pestat] mean and std.dev: (2394.25, 52.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (2108, 2675)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 79.827 CPU sec, 80.887 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (115, 34747, 1236, 56)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1527, 6066, 9522)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 25512)\n",
            "[M::mem_pestat] mean and std.dev: (5393.77, 3354.73)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 33507)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (216, 272, 369)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 675)\n",
            "[M::mem_pestat] mean and std.dev: (263.99, 76.39)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 828)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (715, 2153, 4266)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 11368)\n",
            "[M::mem_pestat] mean and std.dev: (3013.82, 2706.54)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 14919)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2352, 3892, 7757)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 18567)\n",
            "[M::mem_pestat] mean and std.dev: (4736.48, 2913.81)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 23972)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 18.466 CPU sec, 18.595 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (77, 39983, 1002, 107)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2955, 5344, 7014)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15132)\n",
            "[M::mem_pestat] mean and std.dev: (5265.30, 2522.53)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19191)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (222, 296, 757)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 1827)\n",
            "[M::mem_pestat] mean and std.dev: (371.81, 316.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2362)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1753, 4178, 6700)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16594)\n",
            "[M::mem_pestat] mean and std.dev: (4384.20, 2903.18)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21541)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2279, 4524, 7626)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 18320)\n",
            "[M::mem_pestat] mean and std.dev: (4798.71, 2965.98)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 23667)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 24.858 CPU sec, 25.063 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (66, 40003, 1144, 89)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2483, 4833, 7074)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16256)\n",
            "[M::mem_pestat] mean and std.dev: (4776.30, 2711.38)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20847)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (227, 310, 1005)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2561)\n",
            "[M::mem_pestat] mean and std.dev: (494.16, 519.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 3339)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1517, 3541, 6083)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15215)\n",
            "[M::mem_pestat] mean and std.dev: (3965.37, 2801.62)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19781)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1878, 4498, 7563)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 18933)\n",
            "[M::mem_pestat] mean and std.dev: (4684.37, 3009.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 24618)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 24.792 CPU sec, 25.138 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (85, 38659, 1178, 110)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2394, 4713, 8329)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 20199)\n",
            "[M::mem_pestat] mean and std.dev: (4815.59, 3196.66)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 26134)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (224, 299, 712)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 1688)\n",
            "[M::mem_pestat] mean and std.dev: (367.45, 293.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2176)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1618, 3534, 6125)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15139)\n",
            "[M::mem_pestat] mean and std.dev: (4028.20, 2842.17)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19646)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2047, 4171, 7534)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 18508)\n",
            "[M::mem_pestat] mean and std.dev: (4520.00, 2868.49)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 23995)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 27.251 CPU sec, 27.763 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (59, 38691, 1064, 116)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2083, 4152, 7208)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17458)\n",
            "[M::mem_pestat] mean and std.dev: (4654.08, 2792.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22583)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (223, 300, 935)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2359)\n",
            "[M::mem_pestat] mean and std.dev: (444.69, 460.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 3071)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1592, 3737, 6421)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16079)\n",
            "[M::mem_pestat] mean and std.dev: (4128.16, 2849.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20908)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1429, 3202, 6744)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17374)\n",
            "[M::mem_pestat] mean and std.dev: (3990.43, 3024.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22689)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 28.472 CPU sec, 30.170 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (60, 38872, 1045, 133)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1535, 4640, 7007)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17951)\n",
            "[M::mem_pestat] mean and std.dev: (4399.72, 2936.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 23423)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (225, 303, 874)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2172)\n",
            "[M::mem_pestat] mean and std.dev: (424.95, 412.96)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2821)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1660, 3504, 6283)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15529)\n",
            "[M::mem_pestat] mean and std.dev: (4065.58, 2831.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20152)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1294, 3289, 6437)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16723)\n",
            "[M::mem_pestat] mean and std.dev: (4009.66, 2988.26)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21866)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 27.539 CPU sec, 27.868 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (92, 39684, 1228, 124)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2519, 4940, 7597)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17753)\n",
            "[M::mem_pestat] mean and std.dev: (4980.98, 2903.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22831)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (224, 301, 909)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2279)\n",
            "[M::mem_pestat] mean and std.dev: (444.16, 446.95)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2964)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1675, 3630, 6370)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15760)\n",
            "[M::mem_pestat] mean and std.dev: (4059.89, 2781.76)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20455)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1869, 3438, 6369)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15369)\n",
            "[M::mem_pestat] mean and std.dev: (4028.05, 2736.25)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19869)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 25.511 CPU sec, 25.717 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (93, 36776, 1517, 154)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2155, 4183, 7494)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 18172)\n",
            "[M::mem_pestat] mean and std.dev: (4554.31, 2893.31)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 23511)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (223, 302, 802)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 1960)\n",
            "[M::mem_pestat] mean and std.dev: (408.32, 373.24)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2539)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1601, 3525, 6190)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15368)\n",
            "[M::mem_pestat] mean and std.dev: (4006.93, 2797.32)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19957)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1708, 4570, 6974)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17506)\n",
            "[M::mem_pestat] mean and std.dev: (4503.18, 2960.16)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22772)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 28.485 CPU sec, 28.748 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (100, 38470, 1520, 143)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (3647, 5642, 7748)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15950)\n",
            "[M::mem_pestat] mean and std.dev: (5458.67, 2681.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20051)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (224, 302, 779)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 1889)\n",
            "[M::mem_pestat] mean and std.dev: (394.98, 343.45)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2444)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1429, 3277, 5671)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 14155)\n",
            "[M::mem_pestat] mean and std.dev: (3766.77, 2728.55)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 18397)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1926, 4208, 6800)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16548)\n",
            "[M::mem_pestat] mean and std.dev: (4367.73, 2801.91)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21422)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 25.969 CPU sec, 26.288 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (74, 40270, 1142, 98)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1639, 4148, 6721)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16885)\n",
            "[M::mem_pestat] mean and std.dev: (4321.82, 2982.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21967)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (226, 306, 895)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2233)\n",
            "[M::mem_pestat] mean and std.dev: (446.63, 436.78)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2902)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1685, 3577, 6299)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15527)\n",
            "[M::mem_pestat] mean and std.dev: (4061.75, 2793.66)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20141)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1706, 3891, 6591)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16361)\n",
            "[M::mem_pestat] mean and std.dev: (4342.62, 2785.48)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21246)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 25.499 CPU sec, 25.727 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (63, 39754, 979, 108)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (3037, 4638, 7130)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15316)\n",
            "[M::mem_pestat] mean and std.dev: (4851.83, 2783.78)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19409)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (226, 308, 1123)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2917)\n",
            "[M::mem_pestat] mean and std.dev: (538.72, 612.02)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 3814)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1568, 3490, 6372)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15980)\n",
            "[M::mem_pestat] mean and std.dev: (4039.41, 2844.26)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20784)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1738, 3984, 6596)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16312)\n",
            "[M::mem_pestat] mean and std.dev: (4268.02, 2940.49)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21170)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 25.014 CPU sec, 25.204 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (89, 39463, 1301, 119)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2859, 5027, 7468)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16686)\n",
            "[M::mem_pestat] mean and std.dev: (5104.74, 2767.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21295)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (228, 314, 955)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2409)\n",
            "[M::mem_pestat] mean and std.dev: (476.94, 476.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 3136)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1658, 3668, 6229)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15371)\n",
            "[M::mem_pestat] mean and std.dev: (4063.66, 2781.37)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19942)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1626, 3489, 6179)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15285)\n",
            "[M::mem_pestat] mean and std.dev: (4159.87, 2915.65)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19838)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 25.437 CPU sec, 25.663 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (70, 39175, 1200, 107)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (3086, 5133, 8039)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17945)\n",
            "[M::mem_pestat] mean and std.dev: (5097.87, 3084.35)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22898)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (226, 315, 1142)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2974)\n",
            "[M::mem_pestat] mean and std.dev: (565.35, 633.40)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 3890)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1723, 3869, 6443)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15883)\n",
            "[M::mem_pestat] mean and std.dev: (4187.88, 2827.52)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20603)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1623, 3922, 6473)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16173)\n",
            "[M::mem_pestat] mean and std.dev: (4181.23, 2894.21)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21023)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 26.287 CPU sec, 26.608 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (54, 36595, 732, 87)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2782, 5309, 7707)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17557)\n",
            "[M::mem_pestat] mean and std.dev: (5038.81, 2780.62)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22482)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (225, 307, 1216)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3198)\n",
            "[M::mem_pestat] mean and std.dev: (571.41, 679.49)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 4189)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1786, 4162, 6696)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16516)\n",
            "[M::mem_pestat] mean and std.dev: (4361.68, 2883.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21426)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1762, 4417, 6722)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16642)\n",
            "[M::mem_pestat] mean and std.dev: (4424.17, 2873.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21602)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 32.122 CPU sec, 32.516 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (88, 39529, 1160, 78)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2180, 4311, 6953)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16499)\n",
            "[M::mem_pestat] mean and std.dev: (4521.11, 2834.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 21272)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (227, 313, 976)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2474)\n",
            "[M::mem_pestat] mean and std.dev: (482.13, 494.51)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 3223)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1770, 3745, 6518)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16014)\n",
            "[M::mem_pestat] mean and std.dev: (4253.74, 2853.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20762)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1908, 4038, 6578)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15918)\n",
            "[M::mem_pestat] mean and std.dev: (4372.77, 2763.86)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20588)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 24.316 CPU sec, 24.506 real sec\n",
            "[M::process] read 100000 sequences (10000000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (85, 39309, 1238, 110)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (2707, 5021, 7137)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15997)\n",
            "[M::mem_pestat] mean and std.dev: (5069.26, 2631.24)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20427)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (226, 308, 917)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2299)\n",
            "[M::mem_pestat] mean and std.dev: (452.55, 445.69)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2990)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1488, 3316, 6029)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15111)\n",
            "[M::mem_pestat] mean and std.dev: (3843.37, 2750.44)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 19652)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1527, 3104, 5323)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12915)\n",
            "[M::mem_pestat] mean and std.dev: (3705.27, 2782.71)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 16711)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 23.972 CPU sec, 24.190 real sec\n",
            "[M::process] read 12220 sequences (1222000 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (42, 30776, 909, 56)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (3263, 5514, 7982)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17420)\n",
            "[M::mem_pestat] mean and std.dev: (5402.98, 2917.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22139)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (222, 297, 757)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 1827)\n",
            "[M::mem_pestat] mean and std.dev: (379.53, 322.34)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 2362)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1687, 3526, 6350)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 15676)\n",
            "[M::mem_pestat] mean and std.dev: (4021.01, 2818.38)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 20339)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (1648, 3530, 6778)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 17038)\n",
            "[M::mem_pestat] mean and std.dev: (4256.07, 2895.33)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 22168)\n",
            "[M::mem_pestat] skip orientation FF\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_pestat] skip orientation RR\n",
            "[M::mem_process_seqs] Processed 100000 reads in 72.409 CPU sec, 73.312 real sec\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 5761, 33, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (208, 253, 303)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (18, 493)\n",
            "[M::mem_pestat] mean and std.dev: (252.24, 49.76)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 588)\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation RF...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (7610, 7672, 7738)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7354, 7994)\n",
            "[M::mem_pestat] mean and std.dev: (7699.80, 51.01)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (7226, 8122)\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RF\n",
            "[M::mem_process_seqs] Processed 12220 reads in 1.625 CPU sec, 1.582 real sec\n",
            "[main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem MM10 CRISPR.R1.fastq CRISPR.R2.fastq\n",
            "[main] Real time: 5770.285 sec; CPU: 5668.034 sec\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "For the first command, !samtools view converts the alignment SAM file to a binary BAM file. -bS specifies that the input is a SAM file (-S) and the output should be in BAM format (-b). -q 12 filters alignments with mapping quality scores less than 12.\n",
        "\n",
        "For the second command, !sam tools sort sorts the BAM file by genomic coordinates and outputs the sorted alignments to a new BAM file. -o specifies the output where the file is saved."
      ],
      "metadata": {
        "id": "-qqk5GJnsJ7j"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!samtools view -bS -q 12 alignedreads.sam > alignedreads.bam\n",
        "!samtools sort alignedreads.bam -o sortedreads.bam\n"
      ],
      "metadata": {
        "id": "2a-WvhLuUzRP",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6a1150a4-768a-44c3-c0ca-b1cf34af8214"
      },
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[bam_sort_core] merging from 8 files and 1 in-memory blocks...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "!samtools index creates an index file that allows for faster random access to the data in sortedreads.bam."
      ],
      "metadata": {
        "id": "TNOfoQOOtD3d"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!samtools index sortedreads.bam"
      ],
      "metadata": {
        "id": "BNuFM8xHfwR7"
      },
      "execution_count": 11,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "!bcftools mpileup generates a pileup of sequence data from the sorted BAM file (sortedreads.bam), using the reference genome (MM10/Mouse.fasta). -Ou produces an uncompressed binary output, which is used by bcftools call to call variants. -mv specifies that multiallelic and variant calling takes place and -Oz -o causes the output to be a gzipped VCF file: variants.vcf.gz."
      ],
      "metadata": {
        "id": "7SJdLxjitQDw"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!bcftools mpileup -Ou -f MM10/Mouse.fasta sortedreads.bam | bcftools call -mv -Oz -o variants.vcf.gz"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "is5YkKCQf-bR",
        "outputId": "3a28d4b4-114f-47ea-9cb4-f8aabbabd623"
      },
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[mpileup] 1 samples in 1 input files\n",
            "Note: none of --samples-file, --ploidy or --ploidy-file given, assuming all sites are diploid\n",
            "[mpileup] maximum number of reads per input file set to -d 250\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "!pip install cyvcf2 installs the cyvcf2 library, which handles VCF files. I then imported pandas and cyvcf2 to use them. vcf_reader reads the vcf file that is specified by vcf_path, allowing me to extract variant information. I then used \"with open('output.tsv', 'w') as file:\" before a for loop to generate output.tsv. The for loop beneath this command allows me to go over each variant in the VCF file, extracting the chromosome, position, reference allele, alternate allele, quality score, filter status, and genotype. I put these column names into a list, which I used with pd.read_csv to generate a new dataframe titled \"df\"."
      ],
      "metadata": {
        "id": "uXfI1tgWvRin"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install cyvcf2\n",
        "import cyvcf2 as cyvcf2\n",
        "import pandas as pd\n",
        "vcf_path = 'variants.vcf'\n",
        "\n",
        "vcf_reader = cyvcf2.VCF(vcf_path)\n",
        "\n",
        "with open('output.tsv', 'w') as file:\n",
        "    for variant in vcf_reader:\n",
        "        genotype = variant.gt_types[0]\n",
        "        data_string = f\"{variant.CHROM}\\t{variant.POS}\\t{variant.ID}\\t{variant.REF}\\t{','.join(variant.ALT)}\\t{variant.QUAL}\\t{variant.FILTER}\\t{genotype}\\n\"\n",
        "        file.write(data_string)\n",
        "columns = ['CHROM', 'POS', 'ID', 'REF', 'ALT', 'QUAL', 'FILTER', 'GENOTYPE']\n",
        "df = pd.read_csv(\"output.tsv\", delimiter = \"\\t\", header=None, names=columns)"
      ],
      "metadata": {
        "id": "iwln14UHmGJP",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6d82ecc9-9a37-4395-f041-317c2b2c8942"
      },
      "execution_count": 32,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: cyvcf2 in /usr/local/lib/python3.10/dist-packages (0.31.1)\n",
            "Requirement already satisfied: numpy>=1.16.0 in /usr/local/lib/python3.10/dist-packages (from cyvcf2) (1.26.4)\n",
            "Requirement already satisfied: coloredlogs in /usr/local/lib/python3.10/dist-packages (from cyvcf2) (15.0.1)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.10/dist-packages (from cyvcf2) (8.1.7)\n",
            "Requirement already satisfied: humanfriendly>=9.1 in /usr/local/lib/python3.10/dist-packages (from coloredlogs->cyvcf2) (10.0)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "I used pandas to create a new dataframe: intendeddf, which I coded to include dataframe lines that contain 9 specified positions on chromosome 2.\n",
        "\n",
        "This dataframe displays the data of all the intended mutations changed by CRISPR."
      ],
      "metadata": {
        "id": "rN6uVElOwbBJ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "intendeddf = df[(df['CHROM'] == 'chr2') & ((df['POS'] == 36937210) | (df['POS'] == 36996899) | (df['POS'] == 85400441) | (df['POS'] == 85776687) | (df['POS'] == 85918029) | (df['POS'] == 86198668) | (df['POS'] == 86236802) | (df['POS'] == 86658391) | (df['POS'] == 87049235))]\n",
        "intendeddf"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 331
        },
        "id": "_qyndd3NWGLh",
        "outputId": "cbd58ee7-af06-4dfb-f9e9-f1a8e12e55db"
      },
      "execution_count": 39,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "     CHROM       POS     REF      ALT        QUAL  GENOTYPE\n",
              "5109  chr2  36937210       C        T  194.453003         1\n",
              "5110  chr2  36996899  CTTTTT  CTTTTTT  185.158005         3\n",
              "6371  chr2  85400441       A        G  228.171005         3\n",
              "6380  chr2  85776687       T        A  115.749001         1\n",
              "6386  chr2  85918029       T        C  119.155998         1\n",
              "6395  chr2  86198668       A        G  228.250000         3\n",
              "6402  chr2  86236802      CT      CTT  228.218994         3\n",
              "6426  chr2  86658391       C       CG  174.298996         3\n",
              "6437  chr2  87049235       G        T  102.330002         3"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-8060f645-2db7-4af0-8fbd-834a23939be2\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>CHROM</th>\n",
              "      <th>POS</th>\n",
              "      <th>REF</th>\n",
              "      <th>ALT</th>\n",
              "      <th>QUAL</th>\n",
              "      <th>GENOTYPE</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>5109</th>\n",
              "      <td>chr2</td>\n",
              "      <td>36937210</td>\n",
              "      <td>C</td>\n",
              "      <td>T</td>\n",
              "      <td>194.453003</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5110</th>\n",
              "      <td>chr2</td>\n",
              "      <td>36996899</td>\n",
              "      <td>CTTTTT</td>\n",
              "      <td>CTTTTTT</td>\n",
              "      <td>185.158005</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6371</th>\n",
              "      <td>chr2</td>\n",
              "      <td>85400441</td>\n",
              "      <td>A</td>\n",
              "      <td>G</td>\n",
              "      <td>228.171005</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6380</th>\n",
              "      <td>chr2</td>\n",
              "      <td>85776687</td>\n",
              "      <td>T</td>\n",
              "      <td>A</td>\n",
              "      <td>115.749001</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6386</th>\n",
              "      <td>chr2</td>\n",
              "      <td>85918029</td>\n",
              "      <td>T</td>\n",
              "      <td>C</td>\n",
              "      <td>119.155998</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6395</th>\n",
              "      <td>chr2</td>\n",
              "      <td>86198668</td>\n",
              "      <td>A</td>\n",
              "      <td>G</td>\n",
              "      <td>228.250000</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6402</th>\n",
              "      <td>chr2</td>\n",
              "      <td>86236802</td>\n",
              "      <td>CT</td>\n",
              "      <td>CTT</td>\n",
              "      <td>228.218994</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6426</th>\n",
              "      <td>chr2</td>\n",
              "      <td>86658391</td>\n",
              "      <td>C</td>\n",
              "      <td>CG</td>\n",
              "      <td>174.298996</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6437</th>\n",
              "      <td>chr2</td>\n",
              "      <td>87049235</td>\n",
              "      <td>G</td>\n",
              "      <td>T</td>\n",
              "      <td>102.330002</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-8060f645-2db7-4af0-8fbd-834a23939be2')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-8060f645-2db7-4af0-8fbd-834a23939be2 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-8060f645-2db7-4af0-8fbd-834a23939be2');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-23bc5885-4c3d-41cb-a5ad-5d8f29bd45a2\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-23bc5885-4c3d-41cb-a5ad-5d8f29bd45a2')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-23bc5885-4c3d-41cb-a5ad-5d8f29bd45a2 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "  <div id=\"id_2a05b8d2-46be-47fb-98fc-41beffd63c6c\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('intendeddf')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_2a05b8d2-46be-47fb-98fc-41beffd63c6c button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('intendeddf');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "intendeddf",
              "summary": "{\n  \"name\": \"intendeddf\",\n  \"rows\": 9,\n  \"fields\": [\n    {\n      \"column\": \"CHROM\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 1,\n        \"samples\": [\n          \"chr2\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"POS\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 21704765,\n        \"min\": 36937210,\n        \"max\": 87049235,\n        \"num_unique_values\": 9,\n        \"samples\": [\n          86658391\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"REF\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 6,\n        \"samples\": [\n          \"C\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"ALT\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 7,\n        \"samples\": [\n          \"T\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"QUAL\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 51.0969287974667,\n        \"min\": 102.33000183105467,\n        \"max\": 228.25,\n        \"num_unique_values\": 9,\n        \"samples\": [\n          174.2989959716797\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"GENOTYPE\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 1,\n        \"min\": 1,\n        \"max\": 3,\n        \"num_unique_values\": 2,\n        \"samples\": [\n          3\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 39
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "I used panadas to create a new dataframe: offtargetdf, which I used to exclude the targeted variants from the previous dataframe with \"!=\".\n",
        "\n",
        "This dataframe displays the data of all the off-target mutations changed by CRISPR."
      ],
      "metadata": {
        "id": "7ng9OfQpvPJg"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "offtargetdf = df[(df['CHROM'] != 'chr2') & ((df['POS'] != 36937210) | (df['POS'] != 36996899) | (df['POS'] != 85400441) | (df['POS'] != 85776687) | (df['POS'] != 85918029) | (df['POS'] != 86198668) | (df['POS'] != 86236802) | (df['POS'] != 86658391) | (df['POS'] != 87049235))]\n",
        "offtargetdf"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 423
        },
        "id": "RVdxWweVdoaj",
        "outputId": "001e74b1-25eb-4fa0-c731-10a29421bf73"
      },
      "execution_count": 77,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "      CHROM       POS REF ALT      QUAL  GENOTYPE\n",
              "0      chr1   3013991   A   G  11.71720         3\n",
              "1      chr1   3365726   T   A   6.51248         3\n",
              "2      chr1   3408880   G   C   5.75677         3\n",
              "3      chr1   3438070   G   C   3.77660         3\n",
              "4      chr1   3441381   T   A   7.30814         3\n",
              "...     ...       ...  ..  ..       ...       ...\n",
              "56119  chrY  88047416   A   T   5.75677         3\n",
              "56120  chrY  88533987   G   C   8.99921         3\n",
              "56121  chrY  88796561   A   T   5.75677         3\n",
              "56122  chrY  90090135   T   A   5.75677         3\n",
              "56123  chrY  90768233   A   T   3.77660         3\n",
              "\n",
              "[52121 rows x 6 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-8fe547f1-725a-4ef8-8b9a-d08b851999c2\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>CHROM</th>\n",
              "      <th>POS</th>\n",
              "      <th>REF</th>\n",
              "      <th>ALT</th>\n",
              "      <th>QUAL</th>\n",
              "      <th>GENOTYPE</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>chr1</td>\n",
              "      <td>3013991</td>\n",
              "      <td>A</td>\n",
              "      <td>G</td>\n",
              "      <td>11.71720</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>chr1</td>\n",
              "      <td>3365726</td>\n",
              "      <td>T</td>\n",
              "      <td>A</td>\n",
              "      <td>6.51248</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>chr1</td>\n",
              "      <td>3408880</td>\n",
              "      <td>G</td>\n",
              "      <td>C</td>\n",
              "      <td>5.75677</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>chr1</td>\n",
              "      <td>3438070</td>\n",
              "      <td>G</td>\n",
              "      <td>C</td>\n",
              "      <td>3.77660</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>chr1</td>\n",
              "      <td>3441381</td>\n",
              "      <td>T</td>\n",
              "      <td>A</td>\n",
              "      <td>7.30814</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>56119</th>\n",
              "      <td>chrY</td>\n",
              "      <td>88047416</td>\n",
              "      <td>A</td>\n",
              "      <td>T</td>\n",
              "      <td>5.75677</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>56120</th>\n",
              "      <td>chrY</td>\n",
              "      <td>88533987</td>\n",
              "      <td>G</td>\n",
              "      <td>C</td>\n",
              "      <td>8.99921</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>56121</th>\n",
              "      <td>chrY</td>\n",
              "      <td>88796561</td>\n",
              "      <td>A</td>\n",
              "      <td>T</td>\n",
              "      <td>5.75677</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>56122</th>\n",
              "      <td>chrY</td>\n",
              "      <td>90090135</td>\n",
              "      <td>T</td>\n",
              "      <td>A</td>\n",
              "      <td>5.75677</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>56123</th>\n",
              "      <td>chrY</td>\n",
              "      <td>90768233</td>\n",
              "      <td>A</td>\n",
              "      <td>T</td>\n",
              "      <td>3.77660</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>52121 rows × 6 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-8fe547f1-725a-4ef8-8b9a-d08b851999c2')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-8fe547f1-725a-4ef8-8b9a-d08b851999c2 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-8fe547f1-725a-4ef8-8b9a-d08b851999c2');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-a2b07469-25ee-445a-89c4-402d54e6b169\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-a2b07469-25ee-445a-89c4-402d54e6b169')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-a2b07469-25ee-445a-89c4-402d54e6b169 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "  <div id=\"id_f80df482-b72c-4b86-8df2-b54162a823ef\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('offtargetdf')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_f80df482-b72c-4b86-8df2-b54162a823ef button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('offtargetdf');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "offtargetdf",
              "summary": "{\n  \"name\": \"offtargetdf\",\n  \"rows\": 52121,\n  \"fields\": [\n    {\n      \"column\": \"CHROM\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 20,\n        \"samples\": [\n          \"chr1\",\n          \"chr19\",\n          \"chr17\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"POS\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 41554024,\n        \"min\": 259202,\n        \"max\": 195308788,\n        \"num_unique_values\": 52116,\n        \"samples\": [\n          123436035,\n          119516816,\n          61544003\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"REF\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 180,\n        \"samples\": [\n          \"Gaa\",\n          \"Tc\",\n          \"CAAa\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"ALT\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 187,\n        \"samples\": [\n          \"CCT\",\n          \"ACg\",\n          \"CGA\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"QUAL\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 11.716788243919806,\n        \"min\": 3.0488100051879883,\n        \"max\": 228.3300018310547,\n        \"num_unique_values\": 1892,\n        \"samples\": [\n          65.91220092773438,\n          20.96430015563965,\n          134.77099609375\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"GENOTYPE\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0,\n        \"min\": 1,\n        \"max\": 3,\n        \"num_unique_values\": 2,\n        \"samples\": [\n          1,\n          3\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 77
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "I used groupby with 'CHROM' and 'GENOTYPE' to get the amount of mutations and genotypes per chromosome. As the only genotypes were 1 and 3, heterozygous and homozygous mutations, I didn't have to continue editing the dataframe. I then used reset_index() to stop 'CHROM' from being the index and labelled the new columns 'CHROM', 'GENOTYPE', and 'COUNT' with .columns.\n",
        "\n",
        "This dataframe groups by chromosome and genotype, showing the amount of different types of genotype mutations per chromosome. 1 means there was a heterozygous mutation and 3 means there was a homozygous mutation."
      ],
      "metadata": {
        "id": "L_Xck0VBw0dn"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "summary = df.groupby('CHROM')['GENOTYPE'].value_counts()\n",
        "summary_df = summary.reset_index()\n",
        "summary_df.columns = ['CHROM', 'GENOTYPE', 'COUNT']\n",
        "summary"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "pUSLBobNebga",
        "outputId": "39a22c9f-0e19-4708-b075-9332deb87512"
      },
      "execution_count": 75,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "CHROM  GENOTYPE\n",
              "chr1   3           4098\n",
              "       1            266\n",
              "chr10  3           2875\n",
              "       1            196\n",
              "chr11  3           2170\n",
              "       1            209\n",
              "chr12  3           2376\n",
              "       1            119\n",
              "chr13  3           2210\n",
              "       1            187\n",
              "chr14  3           2645\n",
              "       1            184\n",
              "chr15  3           2242\n",
              "       1            124\n",
              "chr16  3           2325\n",
              "       1            139\n",
              "chr17  3           1775\n",
              "       1             99\n",
              "chr18  3           1869\n",
              "       1            146\n",
              "chr19  3           1179\n",
              "       1            109\n",
              "chr2   3           3667\n",
              "       1            336\n",
              "chr3   3           3486\n",
              "       1            134\n",
              "chr4   3           3031\n",
              "       1            217\n",
              "chr5   3           3186\n",
              "       1            180\n",
              "chr6   3           3216\n",
              "       1            180\n",
              "chr7   3           2377\n",
              "       1            295\n",
              "chr8   3           2437\n",
              "       1            113\n",
              "chr9   3           2275\n",
              "       1             84\n",
              "chrX   3           2950\n",
              "       1            263\n",
              "chrY   3            151\n",
              "       1              4\n",
              "Name: count, dtype: int64"
            ],
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th>count</th>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>CHROM</th>\n",
              "      <th>GENOTYPE</th>\n",
              "      <th></th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr1</th>\n",
              "      <th>3</th>\n",
              "      <td>4098</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>266</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr10</th>\n",
              "      <th>3</th>\n",
              "      <td>2875</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>196</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr11</th>\n",
              "      <th>3</th>\n",
              "      <td>2170</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>209</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr12</th>\n",
              "      <th>3</th>\n",
              "      <td>2376</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>119</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr13</th>\n",
              "      <th>3</th>\n",
              "      <td>2210</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>187</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr14</th>\n",
              "      <th>3</th>\n",
              "      <td>2645</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>184</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr15</th>\n",
              "      <th>3</th>\n",
              "      <td>2242</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>124</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr16</th>\n",
              "      <th>3</th>\n",
              "      <td>2325</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>139</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr17</th>\n",
              "      <th>3</th>\n",
              "      <td>1775</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>99</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr18</th>\n",
              "      <th>3</th>\n",
              "      <td>1869</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>146</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr19</th>\n",
              "      <th>3</th>\n",
              "      <td>1179</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>109</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr2</th>\n",
              "      <th>3</th>\n",
              "      <td>3667</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>336</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr3</th>\n",
              "      <th>3</th>\n",
              "      <td>3486</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>134</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr4</th>\n",
              "      <th>3</th>\n",
              "      <td>3031</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>217</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr5</th>\n",
              "      <th>3</th>\n",
              "      <td>3186</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>180</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr6</th>\n",
              "      <th>3</th>\n",
              "      <td>3216</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>180</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr7</th>\n",
              "      <th>3</th>\n",
              "      <td>2377</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>295</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr8</th>\n",
              "      <th>3</th>\n",
              "      <td>2437</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>113</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chr9</th>\n",
              "      <th>3</th>\n",
              "      <td>2275</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>84</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chrX</th>\n",
              "      <th>3</th>\n",
              "      <td>2950</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>263</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th rowspan=\"2\" valign=\"top\">chrY</th>\n",
              "      <th>3</th>\n",
              "      <td>151</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>4</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div><br><label><b>dtype:</b> int64</label>"
            ]
          },
          "metadata": {},
          "execution_count": 75
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "I imported matplotlib so I could use the commands to create a bar graph. I created the labels heterozygous mutation and homozygous mutation and used .groupby to group the summary_df by chromosome and genotype and summed the count values for each chromosome and genotype. unstack() allowed me to reshape the result and create a dataframe suitable for plotting. I then used grouped.plot to plot the data, specifying a bar graph, setting the y label to 'Counts', making the width 0.8, and making the title 'Homozygous and Heterozygous Mutations per Chromosome'. I set the legend with plt.legend using the list of labels created earlier. plt.tight_layout() adjusts the plot layout for better readability, and plt.show() displays the plot.\n",
        "\n",
        "This plot shows the amount of heterozygous and homozygous mutations in each chromosome in a mouse cell culture that went through CRISPR."
      ],
      "metadata": {
        "id": "A3fOYil1w0pb"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import matplotlib.pyplot as plt\n",
        "labels = ['Heterozygous Mutation','Homozygous Mutation']\n",
        "grouped = summary_df.groupby(['CHROM', 'GENOTYPE'])['COUNT'].sum().unstack()\n",
        "ax = grouped.plot(kind='bar', stacked=False, ylabel='Counts', title='Homozygous and Heterozygous Mutations per Chromosome', width=0.8)\n",
        "plt.legend(labels=labels)\n",
        "plt.tight_layout()\n",
        "plt.show()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 487
        },
        "id": "N7pAmudJgk1-",
        "outputId": "180eebcf-d88c-4220-b714-5d12897695a5"
      },
      "execution_count": 74,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 640x480 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAnYAAAHWCAYAAAD6oMSKAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjguMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy81sbWrAAAACXBIWXMAAA9hAAAPYQGoP6dpAAB5eUlEQVR4nO3dd1gU1/s28HtpS5MqVRFQFEVBIzZiRVFUYvSrxliRYFc0dmOiBrvRiL3EqGDsJSYxdlDR2Cv2ggrRqNgFKwic9w9f5ue4gJRlweX+XNdcFztz5jznzMzuPkw5qxBCCBARERHRJ0+nsBtAREREROrBxI6IiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEswsSMiIiLSEkzsiIiIiLQEEzsiIiIiLcHEjugTEx8fD4VCgYiIiMJuClGeRUREQKFQID4+vrCbohVcXFzwxRdfFHYzqAhgYldEZHzInTx5MtPljRo1QpUqVTTcKvqURUdHQ6FQYNOmTZkuDwoKgqmpaZ7q3r59O0JDQ/PROsqvjM8MhUKBgwcPqiwXQsDJyQkKhSLPX/jq2M9TpkzBn3/+ma86irP79+9j+PDhqFixIoyNjWFiYgJvb29MmjQJz549K+zmURHExI6Icm379u0YP358YTeDABgaGmLNmjUq8/fv34///vsPSqUyz3WrYz9nldh169YNr1+/hrOzc77q12YnTpxAlSpVsGDBAtSvXx9hYWGYOXMmPvvsM0ybNg0dOnQo7CZSEaRX2A0gIgLenWF68+YNjIyMCrspn5SWLVti48aNmDt3LvT0/u8jfc2aNfD29sajR48KsXVZ09XVha6ubmE3o1C9fPkSJiYmmS579uwZ/ve//0FXVxdnzpxBxYoVZcsnT56MX3/9tcDbQZ8enrH7hKWmpmLixIkoV64clEolXFxc8P333yM5OVlWLuPei+joaNSoUQNGRkbw9PREdHQ0AGDz5s3w9PSEoaEhvL29cebMGZVYe/fuRf369WFiYgILCwu0bt0aly9flpZn3PeV1QQAP/74I/T19fHw4UOV+nv37g0LCwu8efMGAJCeno7Q0FA4OjrC2NgYvr6+uHTpElxcXBAUFCRb9+bNm/jqq69gZWUFY2Nj1KlTB9u2bZOVyep+nozLlRnbAgBiY2PRrl072Nvbw9DQEKVLl0bHjh2RmJiY7f74559/8NVXX6FMmTJQKpVwcnLCkCFD8Pr1a1m5jEugd+7cQZs2bWBqagobGxsMHz4caWlpsrLPnj1DUFAQzM3NYWFhge7duxf45ZcdO3ZI+7pEiRIICAjAxYsXZe1fsGABAKjsY+Ddvps9ezYqV64MQ0ND2NnZoU+fPnj69KksTsZxuWvXLum4/OWXXwDkbJ+6uLhkebxFR0dj3759UCgU+OOPP1T6uGbNGigUChw5ckSat3HjRnh4eMDQ0BBVqlTBH3/8gaCgILi4uMjWffnyJYYNGwYnJycolUq4u7vj559/hhBCKpPdfZAKhUJ2efP58+cYPHgwXFxcoFQqYWtri6ZNm+L06dNZ7CG5Tp064fHjx4iMjJTmpaSkYNOmTejcubNK+cyO+cza/LH9/PPPP+Pzzz+HtbU1jIyM4O3trXLZX6FQ4OXLl1ixYoW0fsb7N6v35MKFC1G5cmUolUo4OjpiwIABKsd8xq0ply5dgq+vL4yNjVGqVClMnz5dpb/z5s1D5cqVYWxsDEtLS9SoUSPTM5yZbaP169fj+++/h729PUxMTPDll1/i9u3bKuWPHTuG5s2bw9zcHMbGxmjYsCEOHTokKxMaGgqFQoFLly6hc+fOsLS0RL169bJswy+//II7d+4gLCxMJakDADs7O4wZM0Zl/sGDB1GrVi0YGhqibNmy+O2332TLM7b7/v370b9/f9ja2qJ06dLS8txs/3PnzqFhw4YwNjaGm5ubtP/379+P2rVrw8jICO7u7oiKilJp55kzZ9CiRQuYmZnB1NQUTZo0wdGjR2Vl3r59i/Hjx6N8+fIwNDSEtbU16tWrJzvWgY9/PwH/t/2vXbuGrl27wtzcHDY2Nhg7diyEELh9+zZat24NMzMz2NvbY+bMmSptTk5Oxo8//gg3NzfpM37kyJEq37mFTlCREB4eLgCIqKgo8fDhQ5Xp888/F5UrV5at0717dwFAtG/fXixYsEAEBgYKAKJNmzaycs7OzsLd3V04ODiI0NBQMWvWLFGqVClhamoqVq1aJcqUKSOmTZsmpk2bJszNzYWbm5tIS0uT1o+MjBR6enqiQoUKYvr06WL8+PGiZMmSwtLSUsTFxQkhhHjx4oVYuXKlbFq+fLkwNzcXNjY2QgghYmNjBQAxb948WfuSk5OFpaWlCA4OluaNHDlSABCtWrUS8+fPF7169RKlS5cWJUuWFN27d5fKJSQkCDs7O1GiRAnxww8/iLCwMFG1alWho6MjNm/erLJ9M9qbYd++fQKA2Ldvn9QWV1dX4ejoKCZNmiSWLl0qxo8fL2rWrCni4+Oz3YcDBw4ULVu2FFOmTBG//PKL6NGjh9DV1RXt27dX2W+GhoaicuXKIjg4WCxatEi0a9dOABALFy6UyqWnp4sGDRoIHR0d0b9/fzFv3jzRuHFj4eXlJQCI8PDwbNuT0bfly5dnekx17NhRmJiYyNb57bffhEKhEM2bNxfz5s0TP/30k3BxcREWFhbStjt8+LBo2rSpACDb3xl69uwp9PT0RK9evcTixYvFqFGjhImJiahZs6ZISUmRyjk7Ows3NzdhaWkpvvvuO7F48WKxb9++HO/TP/74Q+WYq169utDR0RHnzp0T6enpwsnJSbRr105l27Rs2VKUK1dOer1161ahUCiEl5eXCAsLE2PHjhWWlpaiSpUqwtnZWbZPGjduLBQKhejZs6eYP3++aNWqlQAgBg8eLJWLi4vLch8BED/++KP0unPnzsLAwEAMHTpULF26VPz000+iVatWYtWqVdnu34xj+sSJE+Lzzz8X3bp1k5b9+eefQkdHR9y5c0c4OzuLgIAAadmHx3xWbf7Yfi5durTo37+/mD9/vggLCxO1atUSAMTWrVulMitXrhRKpVLUr19fWv/w4cOy9r//nvzxxx8FAOHn5yfmzZsnQkJChK6ursqx07BhQ+Ho6CicnJzEt99+KxYuXCgaN24sAIjt27dL5ZYsWSJ9Rv7yyy9izpw5okePHmLQoEHZbtuMbeTp6SkdE999950wNDQUFSpUEK9evZLK7tmzRxgYGAgfHx8xc+ZMMWvWLOHl5SUMDAzEsWPHVPrm4eEhWrduLRYuXCgWLFiQZRs+//xzYWRkJJKTk7Nta4aMz3k7Ozvx/fffi/nz54vq1asLhUIhLly4IJXL2O4eHh6iYcOGYt68eWLatGl53v4jRowQ8+bNEx4eHkJXV1esW7dO2Nvbi9DQUDF79mxRqlQpYW5uLpKSkqT1L1y4IExMTISDg4OYOHGimDZtmnB1dRVKpVIcPXpUKvf9998LhUIhevXqJX799Vcxc+ZM0alTJ6m9QuTs++n9vlWrVk106tRJLFy4UAQEBAgAIiwsTLi7u4t+/fqJhQsXirp16woAYv/+/dL6aWlpolmzZsLY2FgMHjxY/PLLLyIkJETo6emJ1q1b52gfaQoTuyIi482W3fR+YhcTEyMAiJ49e8rqGT58uAAg9u7dK81zdnYWAKQPVCGE2LVrlwAgjIyMxL///ivN/+WXX1Q+9KtVqyZsbW3F48ePpXlnz54VOjo6IjAwMMs+9e/fX+jq6sra4uPjI2rXri0rt3nzZlnMhIQEoaenp5KghoaGCgCyxG7w4MECgPjnn3+kec+fPxeurq7CxcVFSlBzmtidOXNGABAbN27Msl9Zef/DPsPUqVOFQqGQbeOMhHzChAmysp999pnw9vaWXv/5558CgJg+fbo0LzU1VdSvXz9XiV120/uJ3fPnz4WFhYXo1auXrJ6EhARhbm4umz9gwACR2f+F//zzjwAgVq9eLZu/c+dOlfkZx+XOnTtlZXO6Tz+0YcMGle06evRooVQqxbNnz6R5Dx48EHp6erLkytPTU5QuXVo8f/5cmhcdHS0AyBK7jH0yadIkWez27dsLhUIhrl+/LoTIXWJnbm4uBgwYkGmfsvN+Yjd//nxRokQJ6Rj86quvhK+vrxBC5DmxEyLr/SyE6vGekpIiqlSpIho3biybb2JiInvPftj+jPfkgwcPhIGBgWjWrJlsH8+fP1/6ByVDw4YNBQDx22+/SfOSk5OFvb29LJFv3bq1yj/EOZGxjUqVKiVLSDKOsTlz5ggh3iX65cuXF/7+/iI9PV0q9+rVK+Hq6iqaNm0qzctILDp16pSjNlhaWoqqVavmuM0Z76cDBw5I8x48eCCUSqUYNmyYNC9ju9erV0+kpqbKyuZ2+69Zs0aad+XKFQFA6OjoyJKzjO+a94+rNm3aCAMDA3Hjxg1p3t27d0WJEiVEgwYNpHlVq1aVHbuZyen3U8b27927tzQvNTVVlC5dWigUClmy+PTpU2FkZCQ7bleuXCl0dHRkn0tCCLF48WIBQBw6dCjbdmoSL8UWMQsWLEBkZKTK5OXlJSu3fft2AMDQoUNl84cNGwYAKpetPDw84OPjI72uXbs2AKBx48YoU6aMyvybN28CAO7du4eYmBgEBQXByspKKufl5YWmTZtK7fjQb7/9hoULF2L69Onw9fWV5gcGBuLYsWO4ceOGNG/16tVwcnJCw4YNAQB79uxBamoq+vfvL6tz4MCBKnG2b9+OWrVqyS5pmJqaonfv3oiPj8elS5cybV9WzM3NAQC7du3Cq1evcrXu+/eGvXz5Eo8ePcLnn38OIUSml7f79u0re12/fn1puwPv+qanp4d+/fpJ83R1dTPdDtkZN25cpsdUs2bNZOUiIyPx7NkzdOrUCY8ePZImXV1d1K5dG/v27ftorI0bN8Lc3BxNmzaV1eHt7Q1TU1OVOlxdXeHv7y+bl5d9eunSJQQHB6N169ayy1OBgYFITk6WXSJcv349UlNT0bVrVwDA3bt3cf78eQQGBsqeEm7YsCE8PT1V2qarq4tBgwbJ5g8bNgxCCOzYseOj2+hDFhYWOHbsGO7evZvrdTN06NABr1+/xtatW/H8+XNs3bo108uw6vT+8f706VMkJiaifv36Ob6E/KGoqCikpKRg8ODB0NH5v6+mXr16wczMTOUzzdTUVNqHAGBgYIBatWrJ3kMWFhb477//cOLEiTy1KTAwECVKlJBet2/fHg4ODtLnXkxMDGJjY9G5c2c8fvxYOt5fvnyJJk2a4MCBA0hPT5fV+eH7PitJSUmy2Dnh4eGB+vXrS69tbGzg7u4u2yYZevXqJbvHMS/bv2PHjtJrd3d3WFhYoFKlStL3CKD6nZKWlobdu3ejTZs2KFu2rFTOwcEBnTt3xsGDB5GUlATg3f67ePEiYmNjM+1vXr6fevbsKf2tq6uLGjVqQAiBHj16SPMtLCxUttvGjRtRqVIlVKxYUfbZ1rhxYwDI0eejpjCxK2Jq1aoFPz8/lcnS0lJW7t9//4WOjg7c3Nxk8+3t7WFhYYF///1XNv/95A34vwTGyckp0/kZ90Nl1OPu7q7S1kqVKkkfYu+LiYlB37590alTJ5XE8+uvv4ZSqcTq1asBAImJidi6dSu6dOki3b+TEfPDvllZWWW6HbJq2/t15ZSrqyuGDh2KpUuXomTJkvD398eCBQs+en8dANy6dUv6gMm4by4jWf1wfUNDQ9jY2MjmWVpayu5D+/fff+Hg4KAyJElm/c2Op6dnpseUg4ODrFzGh2fjxo1hY2Mjm3bv3o0HDx58NFZsbCwSExNha2urUseLFy9U6nB1dVWpI7f7NCkpCW3btkWpUqXw22+/ye4Dq1ixImrWrCkdb8C7fyTq1KkjHV9ZHW+Zzfv333/h6Oio8oWb1+MNAKZPn44LFy7AyckJtWrVQmhoaKZfxNmxsbGBn58f1qxZg82bNyMtLQ3t27fPdVtyY+vWrahTpw4MDQ1hZWUFGxsbLFq0KEfvlcxk9VljYGCAsmXLqmzb0qVLy/Y1oPoeGjVqFExNTVGrVi2UL18eAwYMULn3LTvly5eXvVYoFHBzc5PuC8x4z3Tv3l3leF+6dCmSk5NVtkdmx3xmzMzM8Pz58xy3FVD9nAdUt0lW7VDH9jc3N//od8rDhw/x6tWrLN/j6enp0n2MEyZMwLNnz1ChQgV4enpixIgROHfu3EfbnFFXZt9PmX0XGhoaomTJkirz399usbGxuHjxosp+rlChAgDk6PNRU/hU7CfuwzdWVrJ6+iyr+eK9G8Fz4+nTp2jXrh0qVKiApUuXqiy3tLTEF198gdWrV2PcuHHYtGkTkpOTZf95F4SsttOHDysAwMyZMxEUFIS//voLu3fvxqBBgzB16lQcPXpUdpPxh/U0bdoUT548wahRo1CxYkWYmJjgzp07CAoKUvmvvSg+DZjRxpUrV8Le3l5l+ftPXGZXh62trSyRet+Hyaw6noANCgrC3bt3cfz4cZiZmaksDwwMxLfffov//vsPycnJOHr0KObPn5/vuNnJzfHWoUMH1K9fH3/88Qd2796NGTNm4KeffsLmzZvRokWLHMfs3LkzevXqhYSEBLRo0QIWFhb5bltW/vnnH3z55Zdo0KABFi5cCAcHB+jr6yM8PPyjDyaoS04+uypVqoSrV69i69at2LlzJ37//XcsXLgQ48aNU8twPRnvmRkzZqBatWqZlvnwH7OcHvMVK1ZETEwMUlJSYGBgkKN1cvN5nt/3nia+Uxo0aIAbN25In8VLly7FrFmzsHjxYtmZt9zIrH05aXN6ejo8PT0RFhaWadkPE9rCxMTuE+Xs7Iz09HTExsZKZwuAd4NZPnv2TG1jQ2XUc/XqVZVlV65cQcmSJaXH5NPT09GlSxc8e/YMUVFRMDY2zrTOwMBAtG7dGidOnMDq1avx2WefoXLlyioxr1+/Lvuv8vHjxyr/eTo7O2fZtvfryjjT9+HTXVmdYfH09ISnpyfGjBmDw4cPo27duli8eDEmTZqUafnz58/j2rVrWLFiBQIDA6X5Hz69lRvOzs7Ys2cPXrx4IftyyKy/6lCuXDkAgK2tLfz8/LItm1VyUK5cOURFRaFu3bp5/uLI6T4FgGnTpuHPP//E5s2bM31yEAA6duyIoUOHYu3atXj9+jX09fXx9ddfy+IB7463D304z9nZGVFRUXj+/LnsrF1+jzcHBwf0798f/fv3x4MHD1C9enVMnjw5V4nd//73P/Tp0wdHjx7F+vXrsyyXm7ZltZ9///13GBoaYteuXbJx8sLDw3Ncx4fe/6x5/xJdSkoK4uLiPnpMZsXExARff/01vv76a6SkpKBt27aYPHkyRo8eDUNDw2zX/fASoBAC169fl26NyXjPmJmZ5bl9WWnVqhWOHDmC33//HZ06dVJr3ZkpqO3/IRsbGxgbG2f5HtfR0ZElSVZWVvjmm2/wzTff4MWLF2jQoAFCQ0PRs2fPXH0/5Ve5cuVw9uxZNGnSJMfHdGHhpdhPVMuWLQEAs2fPls3P+G8iICBALXEcHBxQrVo1rFixQvZFcOHCBezevVtqBwCMHz8eu3btwtq1a7O93NCiRQuULFkSP/30E/bv369ytq5JkybQ09PDokWLZPMzO8vSsmVLHD9+XDZsxcuXL7FkyRK4uLjAw8MDwP99AB84cEAql5aWhiVLlsjqS0pKQmpqqmyep6cndHR0sn2kPeM/vvf/wxNCYM6cOVmu8zEtW7ZEamqqbDukpaVh3rx5ea4zO/7+/jAzM8OUKVPw9u1bleXvD1OT8WH5YXLQoUMHpKWlYeLEiSrrp6am5miolpzu06ioKIwZMwY//PAD2rRpk2V9JUuWRIsWLbBq1SqsXr0azZs3l112cXR0RJUqVfDbb7/hxYsX0vz9+/fj/PnzKm1LS0tTORZnzZoFhUIhJWJmZmYoWbKk7HgD3g0l8b60tDSVS3W2trZwdHTM9RAKpqamWLRoEUJDQ9GqVassyzk7O0NXV/ejbQOy3s+6urpQKBSys3zx8fGZDkRsYmKSo/3u5+cHAwMDzJ07V/Y+WrZsGRITE/P0mfb48WPZawMDA3h4eEAIkekx/qHffvtNdjl006ZNuHfvnrSfvb29Ua5cOfz888+yYydDZkM75VTfvn3h4OCAYcOG4dq1ayrLHzx4kOU/mnlRENs/M7q6umjWrBn++usv2VA39+/fx5o1a1CvXj3pzPuH+8/U1BRubm7SeyM330/51aFDB9y5cyfTsQNfv36tcsm3MPGM3SeqatWq6N69O5YsWYJnz56hYcOGOH78OFasWIE2bdrIHljIrxkzZqBFixbw8fFBjx498Pr1a8ybNw/m5ubSeFznz5/HxIkT0aBBAzx48ACrVq2S1fF+8qavr4+OHTti/vz50NXVVflv1M7ODt9++y1mzpyJL7/8Es2bN8fZs2exY8cOlCxZUvbf0nfffYe1a9eiRYsWGDRoEKysrLBixQrExcXh999/l24Crly5MurUqYPRo0fjyZMnsLKywrp161SSuL179yIkJARfffUVKlSogNTUVKxcuRK6urpo165dltuoYsWKKFeuHIYPH447d+7AzMwMv//+e6b3tuRUq1atULduXXz33XeIj4+Hh4cHNm/enOd7mD7GzMwMixYtQrdu3VC9enV07NgRNjY2uHXrFrZt24a6detKCY23tzcAYNCgQfD394euri46duyIhg0bok+fPpg6dSpiYmLQrFkz6OvrIzY2Fhs3bsScOXM+eu9XTvdpp06dYGNjg/Lly6scb02bNoWdnZ30OjAwUIqbWdI5ZcoUtG7dGnXr1sU333yDp0+fYv78+ahSpYrsC7tVq1bw9fXFDz/8gPj4eFStWhW7d+/GX3/9hcGDB0v/QADvbtKeNm0aevbsiRo1auDAgQMqX9DPnz9H6dKl0b59e1StWhWmpqaIiorCiRMnMh1H62O6d+/+0TLm5ub46quvMG/ePCgUCpQrVw5bt27N9B6hrPZzQEAAwsLC0Lx5c3Tu3BkPHjzAggUL4ObmJrsHKqOOqKgohIWFwdHREa6urrKb6zPY2Nhg9OjRGD9+PJo3b44vv/wSV69excKFC1GzZs083a7RrFkz2Nvbo27durCzs8Ply5cxf/58BAQE5OjBBCsrK9SrVw/ffPMN7t+/j9mzZ8PNzQ29evUCAOjo6GDp0qVo0aIFKleujG+++QalSpXCnTt3sG/fPpiZmeHvv//OdbuBd2dW//jjD7Rs2RLVqlVD165dpf1x+vRprF27VvZAXH4VxPbPyqRJkxAZGYl69eqhf//+0NPTwy+//ILk5GTZWIQeHh5o1KgRvL29YWVlhZMnT2LTpk0ICQmRyuTk+0kdunXrhg0bNqBv377Yt28f6tati7S0NFy5cgUbNmyQxuMsEgrjUVxS9f7QBZlp2LChymP7b9++FePHjxeurq5CX19fODk5idGjR4s3b97Iyn043EEGACrDLGQMeTBjxgzZ/KioKFG3bl1hZGQkzMzMRKtWrcSlS5ek5R8bWuNDx48fFwBEs2bNMu1vamqqGDt2rLC3txdGRkaicePG4vLly8La2lr07dtXVvbGjRuiffv2wsLCQhgaGopatWrJxtJ6v5yfn59QKpXSWE+RkZGyoR9u3rwpgoODRbly5YShoaGwsrISvr6+IioqKtN2vu/SpUvCz89PmJqaipIlS4pevXqJs2fPqjzq3717d5Xx44T4v8fx3/f48WPRrVs3YWZmJszNzUW3bt2kIVlyOtxJVkO3ZNWOffv2CX9/f2Fubi4MDQ1FuXLlRFBQkDh58qRUJjU1VQwcOFDY2NgIhUKh0u4lS5YIb29vYWRkJEqUKCE8PT3FyJEjxd27d6UyWR2XQuRsn2Z3vH04lEfGWInm5ubi9evXmcZct26dqFixolAqlaJKlSpiy5Ytol27dqJixYqycs+fPxdDhgwRjo6OQl9fX5QvX17MmDFDNtyFEO+GvOjRo4cwNzcXJUqUEB06dBAPHjyQDXeSnJwsRowYIapWrSpKlCghTExMRNWqVWXjGWblY58ZGTLbzg8fPhTt2rUTxsbGwtLSUvTp00dcuHBB5bjKbj8vW7ZMlC9fXiiVSlGxYkURHh6e6TF85coV0aBBA2FkZCQbriirIYjmz58vKlasKPT19YWdnZ3o16+fePr0qaxMZp+HQrw7pt8fnuaXX34RDRo0ENbW1kKpVIpy5cqJESNGiMTExGy3WcZ7Z+3atWL06NHC1tZWGBkZiYCAANnQRRnOnDkj2rZtK8VxdnYWHTp0EHv27JHKZGybhw8fZhv7Q3fv3hVDhgwRFSpUEIaGhsLY2Fh4e3uLyZMny/qR1fupYcOGomHDhtLrjx03+dn+ufmuOX36tPD39xempqbC2NhY+Pr6yobkEkKISZMmiVq1agkLCwthZGQkKlasKCZPniwbU0+Ij38/CZH19s/qczCzPqakpIiffvpJVK5cWSiVSmFpaSm8vb3F+PHjP3pMaZJCiDzeJU+UD2fPnkW1atXw22+/oVu3bjla59mzZ7C0tMSkSZPwww8/FHALSZukpqbC0dERrVq1wrJly3K8XrVq1WBjY5OveyXp0xMdHQ1fX19s3LixwJ8uJlI33mNHheLXX3+Fqakp2rZtm+nyD3+GC/i/+wkbNWpUgC0jbfTnn3/i4cOHsgdb3vf27VuVy/LR0dE4e/Ysjzci+qTwHjvSqL///huXLl3CkiVLEBISkuUTS+vXr0dERARatmwJU1NTHDx4EGvXrkWzZs1Qt25dDbeaPlXHjh3DuXPnMHHiRHz22WfSuIIfunPnDvz8/NC1a1c4OjriypUrWLx4Mezt7XM8oCwRUVHAxI40auDAgbh//z5atmyZ7ThSXl5e0NPTw/Tp05GUlCQ9UKHOp8BI+y1atAirVq1CtWrVpB+3z4ylpSW8vb2xdOlSPHz4ECYmJggICMC0adNgbW2tuQYTEeUT77EjIiIi0hK8x46IiIhISzCxIyIiItISvMcuB9LT03H37l2UKFGiyP+UCBEREWkXIQSeP38OR0dHaZD2rDCxy4G7d+8WqR/4JSIiouLn9u3bKF26dLZlmNjlQMZPz9y+fVv6DTsiIiIiTUhKSoKTk1OOfgqPiV0OZFx+NTMzY2JHREREhSInt4Px4QkiIiIiLcHEjoiIiEhLMLEjIiIi0hK8x46IiPItLS0Nb9++LexmEH2S9PX1oaurq5a6mNgREVGeCSGQkJCAZ8+eFXZTiD5pFhYWsLe3z/d4uUzsiIgozzKSOltbWxgbG3MQd6JcEkLg1atXePDgAQDAwcEhX/UxsSMiojxJS0uTkjpra+vCbg7RJ8vIyAgA8ODBA9ja2ubrsiwfniAiojzJuKfO2Ni4kFtC9OnLeB/l915VJnZERJQvvPxKlH/qeh8xsSMiIiLSEkzsiIiIiLIRHR0NhULxSTz9zYcniIhI7Vy+26bRePHTAnJVPigoCM+ePcOff/4pmx8dHQ1fX188ffoUFhYWOarLxcUFgwcPxuDBg3PVBvo/oaGhGD9+PPz9/bFz507ZshkzZmDkyJFo2LAhoqOjc1xnXvdLo0aNUK1aNcyePVua9/nnn+PevXswNzfPVV2FgWfsiIiIioCUlJTCbkKhcnBwwL59+/Dff//J5i9fvhxlypQppFa9Y2BgoJYx5jSBiR0REVE2Dh48iPr168PIyAhOTk4YNGgQXr58CeDd2Z1///0XQ4YMgUKhkH3xZ7ce8O6M0sSJExEYGAgzMzP07t0bAPD777+jcuXKUCqVcHFxwcyZM6V1Mi4JfjgFBQUhPj4eOjo6OHnypKz9s2fPhrOzM9LT0wEAW7ZsQfny5WFoaAhfX1+sWLFC5TJjdm0A3t3o/+HZTgsLC0RERAB4l6SGhITAwcEBhoaGcHZ2xtSpU7Pdzra2tmjWrBlWrFghzTt8+DAePXqEgAD5GdlGjRqpnIlr06YNgoKCpOWZ7ZfHjx+jU6dOKFWqFIyNjeHp6Ym1a9dKdQQFBWH//v2YM2eOtF58fHyml2I/to1cXFwwZcoUBAcHo0SJEihTpgyWLFmS7TZQByZ2mhRqnrOJiIiKhBs3bqB58+Zo164dzp07h/Xr1+PgwYMICQkBAGzevBmlS5fGhAkTcO/ePdy7dy9H62X4+eefUbVqVZw5cwZjx47FqVOn0KFDB3Ts2BHnz59HaGgoxo4dKyVMGZcEM6a9e/fC0NAQDRo0gIuLC/z8/BAeHi6LER4ejqCgIOjo6CAuLg7t27dHmzZtcPbsWfTp0wc//PCDrPzH2pATc+fOxZYtW7BhwwZcvXoVq1evhouLy0fXCw4OlsVZvnw5unTpAgMDgxzHBrLeL2/evIG3tze2bduGCxcuoHfv3ujWrRuOHz8OAJgzZw58fHzQq1cvaT0nJyeV+nO6jWbOnIkaNWrgzJkz6N+/P/r164erV6/mqi+5xXvsiIioWNq6dStMTU1l89LS0mSvp06dii5dukhnh8qXL4+5c+eiYcOGWLRoEaysrKCrq4sSJUrA3t4+x+sZGhoCABo3boxhw4ZJ63Xp0gVNmjTB2LFjAQAVKlTApUuXMGPGDAQFBUmXBIF3Z5969uyJ4OBgBAcHAwB69uyJvn37IiwsDEqlEqdPn8b58+fx119/AQB++eUXuLu7Y8aMGQAAd3d3XLhwAZMnT5baEBYWlm0bcuLWrVsoX7486tWrB4VCAWdn5xyt98UXX6Bv3744cOAAvL29sWHDBhw8eBDLly/P0foZstovpUqVwvDhw6XXAwcOxK5du7BhwwbUqlUL5ubmMDAwgLGxsWy9D+V0G7Vs2RL9+/cHAIwaNQqzZs3Cvn374O7unqv+5AbP2BERUbHk6+uLmJgY2bR06VJZmbNnzyIiIgKmpqbS5O/vj/T0dMTFxWVZd07Xq1Gjhmy9y5cvo27durJ5devWRWxsrCzpfPv2Ldq1awdnZ2fMmTNHmt+mTRvo6urijz/+AABERETA19dXOlt29epV1KxZU1Z/rVq18tSG7AQFBSEmJgbu7u4YNGgQdu/enaP19PX10bVrV4SHh2Pjxo2oUKECvLy8crRuTqSlpWHixInw9PSElZUVTE1NsWvXLty6dStX9eR0G73fdoVCAXt7e+mnwwoKz9gREVGxZGJiAjc3N9m8D2/cf/HiBfr06YNBgwaprJ/dDf05Xc/ExCS3zQYA9OvXD7dv38bx48ehp/d/X+UGBgYIDAxEeHg42rZtizVr1sgSP3VRKBQQQsjmvf+LCdWrV0dcXBx27NiBqKgodOjQAX5+fti0adNH6w4ODkbt2rVx4cIF6Uzkh3R0dLKNn5UZM2Zgzpw5mD17Njw9PWFiYoLBgwcX2IMr+vr6stcKhUK617GgFJkzdtOmTYNCoZDdDPnmzRsMGDAA1tbWMDU1Rbt27XD//n3Zerdu3UJAQACMjY1ha2uLESNGIDU1VVYmOjoa1atXh1KphJubW67uEyAiouKrevXquHTpEtzc3FSmjPu+DAwMVM5k5WS9zFSqVAmHDh2SzTt06BAqVKgg/X5oWFgYNmzYgL/++ivT3+jt2bMnoqKisHDhQqSmpqJt27bSMnd3d5WHK06cOJHrNtjY2Ej3rQFAbGwsXr16JVvHzMwMX3/9NX799VesX78ev//+O548eZJl3zNUrlwZlStXxoULF9C5c+dMy3wYPy0tDRcuXJCVyWy/HDp0CK1bt0bXrl1RtWpVlC1bFteuXfvoeh/KyTYqLEUisTtx4gR++eUXldOtQ4YMwd9//42NGzdi//79uHv3ruwATUtLQ0BAAFJSUnD48GGsWLECERERGDdunFQmLi4OAQEB0in3wYMHo2fPnti1a5fG+kdERJ+mUaNG4fDhwwgJCUFMTAxiY2Px119/yR6CcHFxwYEDB3Dnzh08evQox+tlZtiwYdizZw8mTpyIa9euYcWKFZg/f750X1hUVBRGjhyJGTNmoGTJkkhISEBCQgISExOlOipVqoQ6depg1KhR6NSpk/QD8wDQp08fXLlyBaNGjcK1a9ewYcMG6WRHxpOjH2sD8O7ewPnz5+PMmTM4efIk+vbtKzs7FRYWhrVr1+LKlSu4du0aNm7cCHt7+xyPDbh3717cu3cvy/KNGzfGtm3bsG3bNly5cgX9+vVTGTw4s/1Svnx5REZG4vDhw7h8+TL69OmjcsLIxcUFx44dQ3x8PB49epTpGbacbKPCUuiJ3YsXL9ClSxf8+uuvsLS0lOYnJiZi2bJlCAsLQ+PGjeHt7Y3w8HAcPnwYR48eBQDs3r0bly5dwqpVq1CtWjW0aNECEydOxIIFC6TTqosXL4arqytmzpyJSpUqISQkBO3bt8esWbMKpb9ERPTp8PLywv79+3Ht2jXUr18fn332GcaNGwdHR0epzIQJExAfH49y5crBxsYmx+tlpnr16tiwYQPWrVuHKlWqYNy4cZgwYYJ0Q/7BgweRlpaGvn37wsHBQZq+/fZbWT09evRASkqKyqVMV1dXbNq0CZs3b4aXlxcWLVokPRWrVCpz1Abg3dOeTk5OqF+/Pjp37ozhw4dLP2IPACVKlMD06dNRo0YN1KxZE/Hx8di+fTt0dHKWdpiYmGSbBAYHB6N79+4IDAxEw4YNUbZsWfj6+srKZLZfxowZg+rVq8Pf3x+NGjWCvb092rRpI1tv+PDh0NXVhYeHB2xsbDK9/y4n26iwKMSHF6k1rHv37rCyssKsWbNkoz3v3bsXTZo0URn929nZGYMHD8aQIUMwbtw4bNmyBTExMdLyuLg4lC1bFqdPn8Znn32GBg0aoHr16rIRpMPDwzF48GDZfzjZSUpKgrm5ORITE2FmZpb3zuZ0KJPQnLWLiKgwvXnzBnFxcXB1dZWe8qSiYeLEidi4cSPOnTv30bKTJ0/G4sWLcfv2bQ20jLKS3fspN3lIoT48sW7dOpw+fVrl+j4AJCQkwMDAQCVjt7OzQ0JCglTGzs5OZXnGsuzKJCUl4fXr17JT1BmSk5ORnJwsvU5KSsp954iIiDTsxYsXiI+Px/z58zFp0qRMyyxcuBA1a9aEtbU1Dh06hBkzZnz0EjF9OgrtUuzt27fx7bffYvXq1UXuP72pU6fC3NxcmjIbnJCIiKioCQkJgbe3Nxo1apTlE6WxsbFo3bo1PDw8MHHiRAwbNgyhoaGabSgVmEJL7E6dOoUHDx6gevXq0NPTg56eHvbv34+5c+dCT08PdnZ2SElJUbkZ8v79+9Kggfb29io3PWa8/lgZMzOzTM/WAcDo0aORmJgoTTw9TUREn4KIiAgkJydj/fr1WT6dOWvWLNy9exdv3rzBtWvXMHbsWNmQKfRpK7TErkmTJjh//rxsYMgaNWqgS5cu0t/6+vrYs2ePtM7Vq1dx69Yt+Pj4AAB8fHxw/vx52WB/kZGRMDMzg4eHh1Tm/ToyymTUkRmlUgkzMzPZRERERFTUFVqKXqJECVSpUkU2z8TEBNbW1tL8Hj16YOjQobCysoKZmRkGDhwIHx8f1KlTBwDQrFkzeHh4oFu3bpg+fToSEhIwZswYDBgwQHq6p2/fvpg/fz5GjhyJ4OBg7N27Fxs2bMC2bds022EiIiKiAlakz73OmjULOjo6aNeuHZKTk+Hv74+FCxdKy3V1dbF161b069cPPj4+MDExQffu3TFhwgSpjKurK7Zt24YhQ4Zgzpw5KF26NJYuXQp/f//C6BIRERFRgSn04U4+BRzuhIhIFYc7IVIfdQ13UugDFBMRERGRejCxIyIiItISTOyIiIiIAERHR0OhUKgMtfYpKdIPTxAR0Scqp/cUqy1e7u5NDgoKwrNnz/Dnn3/K5kdHR8PX11fl5yxJPUJDQzF+/Hj4+/tj586dsmUzZszAyJEj0bBhQ0RHR+e4ThcXFwwePBiDBw/OVVve/xnTDJ9//jnu3bsHc3MNH79qxDN2REREpDEODg7Yt28f/vvvP9n85cuXo0yZMoXUqncMDAxgb28PhUJRqO3IDyZ2RERE2fj9999RuXJlKJVKuLi4YObMmbLlLi4umDRpEgIDA2FqagpnZ2ds2bIFDx8+ROvWrWFqagovLy+cPHkyx/VmXBL8cAoKCkJ8fDx0dHRU6ps9ezacnZ2Rnp4OANiyZQvKly8PQ0ND+Pr6YsWKFSqXGT/WN4VCoXJW08LCAhEREQCAlJQUhISEwMHBAYaGhnB2dsbUqVOz3Z62trZo1qwZVqxYIc07fPgwHj16hICAAFnZRo0aqZyJa9OmDYKCgqTl//77L4YMGSJtIwB4/PgxOnXqhFKlSsHY2Bienp5Yu3atVEdQUBD279+POXPmSOvFx8dneik2J/t/ypQpCA4ORokSJVCmTBksWbIk221QkJjYERERZeHUqVPo0KEDOnbsiPPnzyM0NBRjx46VEpsMs2bNQt26dXHmzBkEBASgW7duCAwMRNeuXXH69GmUK1cOgYGByBhh7GP1ZlwSzJj27t0LQ0NDNGjQAC4uLvDz80N4eLisDeHh4QgKCoKOjg7i4uLQvn17tGnTBmfPnkWfPn3www8/5Klv2Zk7dy62bNmCDRs24OrVq1i9ejVcXFw+ul5wcLAszvLly9GlSxcYGBjkODYAbN68GaVLl8aECROkbQW8GzrE29sb27Ztw4ULF9C7d29069YNx48fBwDMmTMHPj4+6NWrl7ReZr8Ln9NtNHPmTNSoUQNnzpxB//790a9fP1y9ejVXfVEX3mNHRETF0tatW2Fqaiqbl5aWJnsdFhaGJk2aYOzYsQCAChUq4NKlS5gxY4Z01ggAWrZsiT59+gAAxo0bh0WLFqFmzZr46quvAACjRo2Cj4+P9HvnH6s345Ig8O7sU8+ePREcHIzg4GAAQM+ePdG3b1+EhYVBqVTi9OnTOH/+PP766y8AwC+//AJ3d3fMmDEDAODu7o4LFy5g8uTJue5bdm7duoXy5cujXr16UCgUcHZ2ztF6X3zxBfr27YsDBw7A29sbGzZswMGDB7F8+fIcrZ/BysoKurq6KFGihLS9AKBUqVIYPny49HrgwIHYtWsXNmzYgFq1asHc3BwGBgYwNjaWrfeh3Oz//v37A3i3r2fNmoV9+/bB3d09V/1RB56xIyKiYsnX11f2e+UxMTFYunSprMzly5dRt25d2by6desiNjZWlgR6eXlJf9vZ2QEAPD09VeZl/LZ5Tut9+/Yt2rVrB2dnZ8yZM0ea36ZNG+jq6uKPP/4AAERERMDX11c6W3b16lXUrFlTVn+tWrXy1LfsBAUFISYmBu7u7hg0aBB2796do/X09fXRtWtXhIeHY+PGjahQoYJsG+ZXWloaJk6cCE9PT1hZWcHU1BS7du3CrVu3clVPXva/QqGAvb297HfsNYln7IiIqFgyMTGBm5ubbN6HN/TnlL6+vvR3xn1emc3LuP8tp/r164fbt2/j+PHj0NP7v69sAwMDBAYGIjw8HG3btsWaNWtkiZ+6KBQKfPgDVW/fvpX+rl69OuLi4rBjxw5ERUWhQ4cO8PPzw6ZNmz5ad3BwMGrXro0LFy5IZyI/pKOjk238rMyYMQNz5szB7Nmz4enpCRMTEwwePBgpKSkfXTcv3t/XwLvtltt9rS5M7IiIiLJQqVIlHDp0SDbv0KFDqFChAnR1dQu03rCwMGzYsAGHDx+GtbW1Sh09e/ZElSpVsHDhQqSmpqJt27bSMnd3d2zfvl1W/sSJE7lug42NjXTfGgDExsbi1atXsnXMzMzw9ddf4+uvv0b79u3RvHlzPHnyBFZWVtlug8qVK6Ny5co4d+4cOnfunGmZD+OnpaXhwoUL8PX1leYZGBionGE8dOgQWrduja5duwJ4l1Bfu3YNHh4e2a73oYLa/wWJl2KJiIiyMGzYMOzZswcTJ07EtWvXsGLFCsyfP192/1ZB1BsVFYWRI0dixowZKFmyJBISEpCQkIDExP8br69SpUqoU6cORo0ahU6dOsHIyEha1qdPH1y5cgWjRo3CtWvXsGHDBumG/4yzhznpW+PGjTF//nycOXMGJ0+eRN++fWVnp8LCwrB27VpcuXIF165dw8aNG2Fvb5/jMQD37t2Le/fuZVm+cePG2LZtG7Zt24YrV66gX79+KoMHu7i44MCBA7hz5w4ePXoEAChfvjwiIyNx+PBhXL58GX369MH9+/dV1jt27Bji4+Px6NGjTM+wFdT+L0hM7IiIiLJQvXp1bNiwAevWrUOVKlUwbtw4TJgwIccPF+S13oMHDyItLQ19+/aFg4ODNH377beyenr06IGUlBSVS5murq7YtGkTNm/eDC8vLyxatEh6KlapVOa4bzNnzoSTkxPq16+Pzp07Y/jw4TA2NpaWlyhRAtOnT0eNGjVQs2ZNxMfHY/v27dDRyVl6YWJikm0SGBwcjO7duyMwMBANGzZE2bJlZWfrAGDChAmIj49HuXLlYGNjAwAYM2YMqlevDn9/fzRq1Aj29vZo06aNbL3hw4dDV1cXHh4esLGxyfT+u4La/wVJIT68eE0qkpKSYG5ujsTERJiZmeW9opyOxJ7LEdSJiArDmzdvEBcXB1dXVxgaGhZ2c4qliRMnYuPGjTh37txHy06ePBmLFy/G7du3NdAyyq3s3k+5yUN4jx0REdEn5sWLF4iPj8f8+fMxadKkTMssXLgQNWvWhLW1NQ4dOoQZM2YgJCREwy0lTWNiR0RE9IkJCQnB2rVr0aZNmyyfKI2NjcWkSZPw5MkTlClTBsOGDcPo0aM13FLSNF6KzQFeiiUiUsVLsUTqo65LsXx4goiIiEhLMLEjIiIi0hJM7IiIKF8Ka4R9Im2irvcRH54gIqI8MTAwgI6ODu7evQsbGxsYGBhIg98SUc4IIZCSkoKHDx9CR0cHBgYG+aqPiR0RkTbR4ENaOjo6cHV1xb1793D37t1810dUnBkbG6NMmTI5Htw5K0zsiIgozwwMDFCmTBmkpqZ+9Hc3iShzurq60NPTU8sZbyZ2RESULwqFAvr6+rLfECWiwsGHJ4iIiIi0BBM7IiIiIi3BxI6IiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEsUamK3aNEieHl5wczMDGZmZvDx8cGOHTuk5Y0aNYJCoZBNffv2ldVx69YtBAQEwNjYGLa2thgxYgRSU1NlZaKjo1G9enUolUq4ubkhIiJCE90jIiIi0qhCHaC4dOnSmDZtGsqXLw8hBFasWIHWrVvjzJkzqFy5MgCgV69emDBhgrSOsbGx9HdaWhoCAgJgb2+Pw4cP4969ewgMDIS+vj6mTJkCAIiLi0NAQAD69u2L1atXY8+ePejZsyccHBzg7++v2Q4TERERFaBCTexatWolez158mQsWrQIR48elRI7Y2Nj2NvbZ7r+7t27cenSJURFRcHOzg7VqlXDxIkTMWrUKISGhsLAwACLFy+Gq6srZs6cCQCoVKkSDh48iFmzZjGxIyIiIq1SZO6xS0tLw7p16/Dy5Uv4+PhI81evXo2SJUuiSpUqGD16NF69eiUtO3LkCDw9PWFnZyfN8/f3R1JSEi5evCiV8fPzk8Xy9/fHkSNHsmxLcnIykpKSZBMRERFRUVfovxV7/vx5+Pj44M2bNzA1NcUff/wBDw8PAEDnzp3h7OwMR0dHnDt3DqNGjcLVq1exefNmAEBCQoIsqQMgvU5ISMi2TFJSEl6/fg0jIyOVNk2dOhXjx49Xe1+JiIiIClKhJ3bu7u6IiYlBYmIiNm3ahO7du2P//v3w8PBA7969pXKenp5wcHBAkyZNcOPGDZQrV67A2jR69GgMHTpUep2UlAQnJ6cCi0dE9EkJNc9F2cSCawcRqSj0S7EGBgZwc3ODt7c3pk6diqpVq2LOnDmZlq1duzYA4Pr16wAAe3t73L9/X1Ym43XGfXlZlTEzM8v0bB0AKJVK6UndjImIiIioqCv0xO5D6enpSE5OznRZTEwMAMDBwQEA4OPjg/Pnz+PBgwdSmcjISJiZmUmXc318fLBnzx5ZPZGRkbL7+IiIiIi0QaFeih09ejRatGiBMmXK4Pnz51izZg2io6Oxa9cu3LhxA2vWrEHLli1hbW2Nc+fOYciQIWjQoAG8vLwAAM2aNYOHhwe6deuG6dOnIyEhAWPGjMGAAQOgVCoBAH379sX8+fMxcuRIBAcHY+/evdiwYQO2bdtWmF0nIiIiUrtCTewePHiAwMBA3Lt3D+bm5vDy8sKuXbvQtGlT3L59G1FRUZg9ezZevnwJJycntGvXDmPGjJHW19XVxdatW9GvXz/4+PjAxMQE3bt3l4175+rqim3btmHIkCGYM2cOSpcujaVLl3KoEyIiItI6CiGEKOxGFHVJSUkwNzdHYmJi/u63y+kNx7zZmIjyShOfM3x4gkijcpOHFLl77IiIiIgob5jYEREREWkJJnZEREREWoKJHREREZGWYGJHREREpCWY2BERERFpCSZ2RERERFqCiR0RERGRlmBiR0RERKQlmNgRERERaQkmdkRERERagokdERERkZZgYkdERESkJZjYEREREWkJJnZEREREWoKJHREREZGWYGJHREREpCX0CrsBREREWi/UPBdlEwuuHaT1eMaOiIiISEvwjB0RERVvOT2bxjNp9AngGTsiIiIiLcHEjoiIiEhLMLEjIiIi0hJM7IiIiIi0BBM7IiIiIi3BxI6IiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEsUamK3aNEieHl5wczMDGZmZvDx8cGOHTuk5W/evMGAAQNgbW0NU1NTtGvXDvfv35fVcevWLQQEBMDY2Bi2trYYMWIEUlNTZWWio6NRvXp1KJVKuLm5ISIiQhPdIyIiItKoQk3sSpcujWnTpuHUqVM4efIkGjdujNatW+PixYsAgCFDhuDvv//Gxo0bsX//fty9exdt27aV1k9LS0NAQABSUlJw+PBhrFixAhERERg3bpxUJi4uDgEBAfD19UVMTAwGDx6Mnj17YteuXRrvLxEREVFB0ivM4K1atZK9njx5MhYtWoSjR4+idOnSWLZsGdasWYPGjRsDAMLDw1GpUiUcPXoUderUwe7du3Hp0iVERUXBzs4O1apVw8SJEzFq1CiEhobCwMAAixcvhqurK2bOnAkAqFSpEg4ePIhZs2bB399f430mIqIcCjXPYbnEgm0H0SekyNxjl5aWhnXr1uHly5fw8fHBqVOn8PbtW/j5+UllKlasiDJlyuDIkSMAgCNHjsDT0xN2dnZSGX9/fyQlJUln/Y4cOSKrI6NMRh1ERERE2qJQz9gBwPnz5+Hj44M3b97A1NQUf/zxBzw8PBATEwMDAwNYWFjIytvZ2SEhIQEAkJCQIEvqMpZnLMuuTFJSEl6/fg0jIyOVNiUnJyM5OVl6nZSUlO9+EhERERW0Qj9j5+7ujpiYGBw7dgz9+vVD9+7dcenSpUJt09SpU2Fubi5NTk5OhdoeIiIiopwo9MTOwMAAbm5u8Pb2xtSpU1G1alXMmTMH9vb2SElJwbNnz2Tl79+/D3t7ewCAvb29ylOyGa8/VsbMzCzTs3UAMHr0aCQmJkrT7du31dFVIiIiogJV6Indh9LT05GcnAxvb2/o6+tjz5490rKrV6/i1q1b8PHxAQD4+Pjg/PnzePDggVQmMjISZmZm8PDwkMq8X0dGmYw6MqNUKqUhWDImIiIioqKuUO+xGz16NFq0aIEyZcrg+fPnWLNmDaKjo7Fr1y6Ym5ujR48eGDp0KKysrGBmZoaBAwfCx8cHderUAQA0a9YMHh4e6NatG6ZPn46EhASMGTMGAwYMgFKpBAD07dsX8+fPx8iRIxEcHIy9e/diw4YN2LZtW2F2nYiIiEjtCjWxe/DgAQIDA3Hv3j2Ym5vDy8sLu3btQtOmTQEAs2bNgo6ODtq1a4fk5GT4+/tj4cKF0vq6urrYunUr+vXrBx8fH5iYmKB79+6YMGGCVMbV1RXbtm3DkCFDMGfOHJQuXRpLly7lUCdERESkdcPqFGpit2zZsmyXGxoaYsGCBViwYEGWZZydnbF9+/Zs62nUqBHOnDmTpzYSERERfSqK3D12RERERJQ3TOyIiIiItAQTOyIiIiItwcSOiIiISEsU+k+KUQHQsid8iIiIKGd4xo6IiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEswsSMiIiLSEkzsiIiIiLQEEzsiIiIiLcHEjoiIiEhLMLEjIiIi0hJM7IiIiIi0BBM7IiIiIi3BxI6IiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEswsSMiIiLSEnqF3QCiYiPUPIflEgu2HUREpLV4xo6IiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEswsSMiIiLSEkzsiIiIiLREoSZ2U6dORc2aNVGiRAnY2tqiTZs2uHr1qqxMo0aNoFAoZFPfvn1lZW7duoWAgAAYGxvD1tYWI0aMQGpqqqxMdHQ0qlevDqVSCTc3N0RERBR094iIiIg0qlATu/3792PAgAE4evQoIiMj8fbtWzRr1gwvX76UlevVqxfu3bsnTdOnT5eWpaWlISAgACkpKTh8+DBWrFiBiIgIjBs3TioTFxeHgIAA+Pr6IiYmBoMHD0bPnj2xa9cujfWViIiIqKAV6i9P7Ny5U/Y6IiICtra2OHXqFBo0aCDNNzY2hr29faZ17N69G5cuXUJUVBTs7OxQrVo1TJw4EaNGjUJoaCgMDAywePFiuLq6YubMmQCASpUq4eDBg5g1axb8/f0LroNEREREGlSk7rFLTHz3U0pWVlay+atXr0bJkiVRpUoVjB49Gq9evZKWHTlyBJ6enrCzs5Pm+fv7IykpCRcvXpTK+Pn5yer09/fHkSNHCqorRERERBpXZH4rNj09HYMHD0bdunVRpUoVaX7nzp3h7OwMR0dHnDt3DqNGjcLVq1exefNmAEBCQoIsqQMgvU5ISMi2TFJSEl6/fg0jIyPZsuTkZCQnJ0uvk5KS1NdRIiIiogJSZBK7AQMG4MKFCzh48KBsfu/evaW/PT094eDggCZNmuDGjRsoV65cgbRl6tSpGD9+fIHUTURERFRQisSl2JCQEGzduhX79u1D6dKlsy1bu3ZtAMD169cBAPb29rh//76sTMbrjPvysipjZmamcrYOAEaPHo3ExERpun37dt46RkRERKRBhZrYCSEQEhKCP/74A3v37oWrq+tH14mJiQEAODg4AAB8fHxw/vx5PHjwQCoTGRkJMzMzeHh4SGX27NkjqycyMhI+Pj6ZxlAqlTAzM5NNREREREVdoSZ2AwYMwKpVq7BmzRqUKFECCQkJSEhIwOvXrwEAN27cwMSJE3Hq1CnEx8djy5YtCAwMRIMGDeDl5QUAaNasGTw8PNCtWzecPXsWu3btwpgxYzBgwAAolUoAQN++fXHz5k2MHDkSV65cwcKFC7FhwwYMGTKk0PpOREREpG6FmtgtWrQIiYmJaNSoERwcHKRp/fr1AAADAwNERUWhWbNmqFixIoYNG4Z27drh77//lurQ1dXF1q1boaurCx8fH3Tt2hWBgYGYMGGCVMbV1RXbtm1DZGQkqlatipkzZ2Lp0qUc6oSIiIi0SqE+PCGEyHa5k5MT9u/f/9F6nJ2dsX379mzLNGrUCGfOnMlV+4iIiCgToeY5LJdYsO0gFUXi4QkiIiIiyr88JXanT5/G+fPnpdd//fUX2rRpg++//x4pKSlqaxwRERER5VyeErs+ffrg2rVrAICbN2+iY8eOMDY2xsaNGzFy5Ei1NpCIiIiIciZPid21a9dQrVo1AMDGjRvRoEEDrFmzBhEREfj999/V2T4iIiIiyqE8PTwhhEB6ejoAICoqCl988QWAdw87PHr0SH2tI+INukREOcfPzGIvT2fsatSogUmTJmHlypXYv38/AgICAABxcXEqv8lKRERERJqRp8Ru1qxZOH36NEJCQvDDDz/Azc0NALBp0yZ8/vnnam0gEREREeVMni7FVq1aVfZUbIYZM2ZAT69Qh8YjIiIiKrbylIWVLVsWJ06cgLW1tWz+mzdvUL16ddy8eVMtjSOiIiqn9/EAvJeHiEiD8nQpNj4+HmlpaSrzk5OT8d9//+W7UURERESUe7k6Y7dlyxbp7127dsHc/P/+a09LS8OePXvg6uqqvtYRERERUY7lKrFr06YNAEChUKB79+6yZfr6+nBxccHMmTPV1jgiIiIiyrlcJXYZY9e5urrixIkTKFmyZIE0ioiIiIo5jsmXJ3l6eCIuLk7d7SAiIiKifMrz2CR79uzBnj178ODBA+lMXobly5fnu2FERERElDt5SuzGjx+PCRMmoEaNGnBwcIBCoVB3u4gor3j5goio2MpTYrd48WJERESgW7du6m4PEREREeVRnsaxS0lJ4U+HERERERUxeTpj17NnT6xZswZjx45Vd3uICgcvXxIRkRbIU2L35s0bLFmyBFFRUfDy8oK+vr5seVhYmFoaR0REREQ5l6fE7ty5c6hWrRoA4MKFC7JlfJCCiIiIqHDkKbHbt2+futtBnxr+CDwREVGRk6eHJ4iIiIio6MnTGTtfX99sL7nu3bs3zw0iIiIiorzJU2KXcX9dhrdv3yImJgYXLlxA9+7d1dEuIiIiIsqlPCV2s2bNynR+aGgoXrx4ka8GEREREVHeqPUeu65du/J3YomIiIgKiVoTuyNHjsDQ0FCdVRIRERFRDuXpUmzbtm1lr4UQuHfvHk6ePMlfoyAiIiIqJHlK7MzN5WOY6ejowN3dHRMmTECzZs3U0jAiIiIiyp08JXbh4eHqbgcRERER5VO+7rE7deoUVq1ahVWrVuHMmTO5Xn/q1KmoWbMmSpQoAVtbW7Rp0wZXr16VlXnz5g0GDBgAa2trmJqaol27drh//76szK1btxAQEABjY2PY2tpixIgRSE1NlZWJjo5G9erVoVQq4ebmhoiIiFy3l4iIiKgoy1Ni9+DBAzRu3Bg1a9bEoEGDMGjQIHh7e6NJkyZ4+PBhjuvZv38/BgwYgKNHjyIyMhJv375Fs2bN8PLlS6nMkCFD8Pfff2Pjxo3Yv38/7t69K7vHLy0tDQEBAUhJScHhw4exYsUKREREYNy4cVKZuLg4BAQEwNfXFzExMRg8eDB69uyJXbt25aX7REREREVSnhK7gQMH4vnz57h48SKePHmCJ0+e4MKFC0hKSsKgQYNyXM/OnTsRFBSEypUro2rVqoiIiMCtW7dw6tQpAEBiYiKWLVuGsLAwNG7cGN7e3ggPD8fhw4dx9OhRAMDu3btx6dIlrFq1CtWqVUOLFi0wceJELFiwACkpKQCAxYsXw9XVFTNnzkSlSpUQEhKC9u3bZzkeHxEREdGnKE+J3c6dO7Fw4UJUqlRJmufh4YEFCxZgx44deW5MYuK7H4u3srIC8O5S79u3b+Hn5yeVqVixIsqUKYMjR44AeDfEiqenJ+zs7KQy/v7+SEpKwsWLF6Uy79eRUSajjg8lJycjKSlJNhEREREVdXlK7NLT06Gvr68yX19fH+np6XlqSHp6OgYPHoy6deuiSpUqAICEhAQYGBjAwsJCVtbOzg4JCQlSmfeTuozlGcuyK5OUlITXr1+rtGXq1KkwNzeXJicnpzz1iYiIiEiT8pTYNW7cGN9++y3u3r0rzbtz5w6GDBmCJk2a5KkhAwYMwIULF7Bu3bo8ra9Oo0ePRmJiojTdvn27sJtERERE9FF5Suzmz5+PpKQkuLi4oFy5cihXrhxcXV2RlJSEefPm5bq+kJAQbN26Ffv27UPp0qWl+fb29khJScGzZ89k5e/fvw97e3upzIdPyWa8/lgZMzMzGBkZqbRHqVTCzMxMNhEREREVdXkax87JyQmnT59GVFQUrly5AgCoVKmSyn1sHyOEwMCBA/HHH38gOjoarq6usuXe3t7Q19fHnj170K5dOwDA1atXcevWLfj4+AAAfHx8MHnyZDx48AC2trYAgMjISJiZmcHDw0Mqs337dlndkZGRUh1ERERE2iBXZ+z27t0LDw8PJCUlQaFQoGnTphg4cCAGDhyImjVronLlyvjnn39yXN+AAQOwatUqrFmzBiVKlEBCQgISEhKk+97Mzc3Ro0cPDB06FPv27cOpU6fwzTffwMfHB3Xq1AEANGvWDB4eHujWrRvOnj2LXbt2YcyYMRgwYACUSiUAoG/fvrh58yZGjhyJK1euYOHChdiwYQOGDBmSm+4TERERFWm5Suxmz56NXr16ZXpp0tzcHH369EFYWFiO61u0aBESExPRqFEjODg4SNP69eulMrNmzcIXX3yBdu3aoUGDBrC3t8fmzZul5bq6uti6dSt0dXXh4+ODrl27IjAwEBMmTJDKuLq6Ytu2bYiMjETVqlUxc+ZMLF26FP7+/rnpPhEREVGRlqtLsWfPnsVPP/2U5fJmzZrh559/znF9QoiPljE0NMSCBQuwYMGCLMs4OzurXGr9UKNGjfL06xhEREREn4pcnbG7f/9+psOcZNDT08vVL08QERERkfrkKrErVaoULly4kOXyc+fOwcHBId+NIiIiIqLcy1Vi17JlS4wdOxZv3rxRWfb69Wv8+OOP+OKLL9TWOCIiIiLKuVzdYzdmzBhs3rwZFSpUQEhICNzd3QEAV65cwYIFC5CWloYffvihQBpKRERERNnLVWJnZ2eHw4cPo1+/fhg9erT08INCoYC/vz8WLFig8tNdRERFXqh5DsslFmw7iIjyKdcDFGc8gfr06VNcv34dQgiUL18elpaWBdE+IiIiIsqhPP3yBABYWlqiZs2a6mwLEREREeVDnn4rloiIiIiKHiZ2RERERFqCiR0RERGRlmBiR0RERKQlmNgRERERaYk8PxVLRFTgOL4cEVGu8IwdERERkZZgYkdERESkJZjYEREREWkJJnZEREREWoKJHREREZGWYGJHREREpCWY2BERERFpCSZ2RERERFqCiR0RERGRlmBiR0RERKQlmNgRERERaQkmdkRERERagokdERERkZZgYkdERESkJZjYEREREWkJJnZEREREWoKJHREREZGWKNTE7sCBA2jVqhUcHR2hUCjw559/ypYHBQVBoVDIpubNm8vKPHnyBF26dIGZmRksLCzQo0cPvHjxQlbm3LlzqF+/PgwNDeHk5ITp06cXdNeIiIiINK5QE7uXL1+iatWqWLBgQZZlmjdvjnv37knT2rVrZcu7dOmCixcvIjIyElu3bsWBAwfQu3dvaXlSUhKaNWsGZ2dnnDp1CjNmzEBoaCiWLFlSYP0iIiIiKgx6hRm8RYsWaNGiRbZllEol7O3tM112+fJl7Ny5EydOnECNGjUAAPPmzUPLli3x888/w9HREatXr0ZKSgqWL18OAwMDVK5cGTExMQgLC5MlgERERESfuiJ/j110dDRsbW3h7u6Ofv364fHjx9KyI0eOwMLCQkrqAMDPzw86Ojo4duyYVKZBgwYwMDCQyvj7++Pq1at4+vSp5jpCREREVMAK9YzdxzRv3hxt27aFq6srbty4ge+//x4tWrTAkSNHoKuri4SEBNja2srW0dPTg5WVFRISEgAACQkJcHV1lZWxs7OTlllaWqrETU5ORnJysvQ6KSlJ3V0jIiIiUrsindh17NhR+tvT0xNeXl4oV64coqOj0aRJkwKLO3XqVIwfP77A6iciIiIqCEX+Uuz7ypYti5IlS+L69esAAHt7ezx48EBWJjU1FU+ePJHuy7O3t8f9+/dlZTJeZ3Xv3ujRo5GYmChNt2/fVndXiIiIiNTuk0rs/vvvPzx+/BgODg4AAB8fHzx79gynTp2Syuzduxfp6emoXbu2VObAgQN4+/atVCYyMhLu7u6ZXoYF3j2wYWZmJpuIiIiIirpCTexevHiBmJgYxMTEAADi4uIQExODW7du4cWLFxgxYgSOHj2K+Ph47NmzB61bt4abmxv8/f0BAJUqVULz5s3Rq1cvHD9+HIcOHUJISAg6duwIR0dHAEDnzp1hYGCAHj164OLFi1i/fj3mzJmDoUOHFla3iYiIiApEoSZ2J0+exGeffYbPPvsMADB06FB89tlnGDduHHR1dXHu3Dl8+eWXqFChAnr06AFvb2/8888/UCqVUh2rV69GxYoV0aRJE7Rs2RL16tWTjVFnbm6O3bt3Iy4uDt7e3hg2bBjGjRvHoU6IiIhI6xTqwxONGjWCECLL5bt27fpoHVZWVlizZk22Zby8vPDPP//kun1EREREn5JP6h47IiIiIsoaEzsiIiIiLcHEjoiIiEhLMLEjIiIi0hJM7IiIiIi0BBM7IiIiIi3BxI6IiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEswsSMiIiLSEkzsiIiIiLQEEzsiIiIiLcHEjoiIiEhLMLEjIiIi0hJM7IiIiIi0BBM7IiIiIi3BxI6IiIhISzCxIyIiItISTOyIiIiItIReYTeAiKhYCDXPYbnEgm0HEWk1nrEjIiIi0hJM7IiIiIi0BBM7IiIiIi3BxI6IiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEswsSMiIiLSEoWa2B04cACtWrWCo6MjFAoF/vzzT9lyIQTGjRsHBwcHGBkZwc/PD7GxsbIyT548QZcuXWBmZgYLCwv06NEDL168kJU5d+4c6tevD0NDQzg5OWH69OkF3TUiIiIijSvUxO7ly5eoWrUqFixYkOny6dOnY+7cuVi8eDGOHTsGExMT+Pv7482bN1KZLl264OLFi4iMjMTWrVtx4MAB9O7dW1qelJSEZs2awdnZGadOncKMGTMQGhqKJUuWFHj/iIiIiDSpUH8rtkWLFmjRokWmy4QQmD17NsaMGYPWrVsDAH777TfY2dnhzz//RMeOHXH58mXs3LkTJ06cQI0aNQAA8+bNQ8uWLfHzzz/D0dERq1evRkpKCpYvXw4DAwNUrlwZMTExCAsLkyWARERERJ+6InuPXVxcHBISEuDn5yfNMzc3R+3atXHkyBEAwJEjR2BhYSEldQDg5+cHHR0dHDt2TCrToEEDGBgYSGX8/f1x9epVPH36VEO9ISIiIip4hXrGLjsJCQkAADs7O9l8Ozs7aVlCQgJsbW1ly/X09GBlZSUr4+rqqlJHxjJLS0uV2MnJyUhOTpZeJyUl5bM3RERERAWvyJ6xK0xTp06Fubm5NDk5ORV2k4iIiIg+qsgmdvb29gCA+/fvy+bfv39fWmZvb48HDx7IlqempuLJkyeyMpnV8X6MD40ePRqJiYnSdPv27fx3iIiIiKiAFdnEztXVFfb29tizZ480LykpCceOHYOPjw8AwMfHB8+ePcOpU6ekMnv37kV6ejpq164tlTlw4ADevn0rlYmMjIS7u3uml2EBQKlUwszMTDYRERERFXWFmti9ePECMTExiImJAfDugYmYmBjcunULCoUCgwcPxqRJk7BlyxacP38egYGBcHR0RJs2bQAAlSpVQvPmzdGrVy8cP34chw4dQkhICDp27AhHR0cAQOfOnWFgYIAePXrg4sWLWL9+PebMmYOhQ4cWUq+JiIiICkahPjxx8uRJ+Pr6Sq8zkq3u3bsjIiICI0eOxMuXL9G7d288e/YM9erVw86dO2FoaCits3r1aoSEhKBJkybQ0dFBu3btMHfuXGm5ubk5du/ejQEDBsDb2xslS5bEuHHjONQJERERaZ1CTewaNWoEIUSWyxUKBSZMmIAJEyZkWcbKygpr1qzJNo6Xlxf++eefPLeTiIiI6FNQZO+xIyIiIqLcYWJHREREpCWY2BERERFpCSZ2RERERFqCiR0RERGRlmBiR0RERKQlmNgRERERaQkmdkRERERagokdERERkZZgYkdERESkJZjYEREREWkJJnZEREREWoKJHREREZGWYGJHREREpCWY2BERERFpCSZ2RERERFqCiR0RERGRlmBiR0RERKQlmNgRERERaQkmdkRERERagokdERERkZZgYkdERESkJZjYEREREWkJJnZEREREWoKJHREREZGWYGJHREREpCWY2BERERFpCSZ2RERERFqCiR0RERGRlmBiR0RERKQlmNgRERERaYkindiFhoZCoVDIpooVK0rL37x5gwEDBsDa2hqmpqZo164d7t+/L6vj1q1bCAgIgLGxMWxtbTFixAikpqZquitEREREBU6vsBvwMZUrV0ZUVJT0Wk/v/5o8ZMgQbNu2DRs3boS5uTlCQkLQtm1bHDp0CACQlpaGgIAA2Nvb4/Dhw7h37x4CAwOhr6+PKVOmaLwvRERERAWpyCd2enp6sLe3V5mfmJiIZcuWYc2aNWjcuDEAIDw8HJUqVcLRo0dRp04d7N69G5cuXUJUVBTs7OxQrVo1TJw4EaNGjUJoaCgMDAw03R0iIiKiAlOkL8UCQGxsLBwdHVG2bFl06dIFt27dAgCcOnUKb9++hZ+fn1S2YsWKKFOmDI4cOQIAOHLkCDw9PWFnZyeV8ff3R1JSEi5evJhlzOTkZCQlJckmIiIioqKuSCd2tWvXRkREBHbu3IlFixYhLi4O9evXx/Pnz5GQkAADAwNYWFjI1rGzs0NCQgIAICEhQZbUZSzPWJaVqVOnwtzcXJqcnJzU2zEiIiKiAlCkL8W2aNFC+tvLywu1a9eGs7MzNmzYACMjowKLO3r0aAwdOlR6nZSUxOSOiIiIirwifcbuQxYWFqhQoQKuX78Oe3t7pKSk4NmzZ7Iy9+/fl+7Js7e3V3lKNuN1ZvftZVAqlTAzM5NNREREREXdJ5XYvXjxAjdu3ICDgwO8vb2hr6+PPXv2SMuvXr2KW7duwcfHBwDg4+OD8+fP48GDB1KZyMhImJmZwcPDQ+PtJyIiIipIRfpS7PDhw9GqVSs4Ozvj7t27+PHHH6Grq4tOnTrB3NwcPXr0wNChQ2FlZQUzMzMMHDgQPj4+qFOnDgCgWbNm8PDwQLdu3TB9+nQkJCRgzJgxGDBgAJRKZSH3joiIiEi9inRi999//6FTp054/PgxbGxsUK9ePRw9ehQ2NjYAgFmzZkFHRwft2rVDcnIy/P39sXDhQml9XV1dbN26Ff369YOPjw9MTEzQvXt3TJgwobC6RERERFRginRit27dumyXGxoaYsGCBViwYEGWZZydnbF9+3Z1N42IiIioyPmk7rEjIiIioqwxsSMiIiLSEkzsiIiIiLQEEzsiIiIiLcHEjoiIiEhLMLEjIiIi0hJM7IiIiIi0RJEex46IiN5x+W5bjsrFGxZwQ4ioSOMZOyIiIiItwcSOiIiISEvwUiwREVERx0vxlFM8Y0dERESkJZjYEREREWkJXopVA54iJyIiKjpy+r0MaN93M8/YEREREWkJJnZEREREWoKJHREREZGW4D12JMP7BYmooPFzhqjg8IwdERERkZbgGbtPRHF+wievuM2IiKi4YWJHlE+8rERU9GjqHzu+/6mo4aVYIiIiIi3BM3ZEREREBSnUPBdlE/MViokdEUm06bIS77EkouKIiR0REQHQrsSeqLhiYkeFgl8gucOzT0RU0Pg5ox2Y2BER5QP/SSGiooSJHRFpHJMhIqKCwcSOiIiINIb/2BUsjmNHREREpCWKVWK3YMECuLi4wNDQELVr18bx48cLu0lEREREalNsErv169dj6NCh+PHHH3H69GlUrVoV/v7+ePDgQWE3jYiIiEgtik1iFxYWhl69euGbb76Bh4cHFi9eDGNjYyxfvrywm0ZERESkFsUisUtJScGpU6fg5+cnzdPR0YGfnx+OHDlSiC0jIiIiUp9i8VTso0ePkJaWBjs7O9l8Ozs7XLlyRaV8cnIykpOTpdeJie9+ty0pKSnT+tOTX+WoHUkKkbMGZxInpzE0FSfHMTQVR5u2mTb1RVNxtGmbfQKfM9z/uY/DbZb7OJ9CX3IsOX/7PyP/ECIH9Yhi4M6dOwKAOHz4sGz+iBEjRK1atVTK//jjjwIAJ06cOHHixIlTkZlu37790ZynWJyxK1myJHR1dXH//n3Z/Pv378Pe3l6l/OjRozF06FDpdXp6Op48eQJra2soFIo8tSEpKQlOTk64ffs2zMzM8lRHUYmjTX3RVBz2pXjHYV+Kdxxt6oum4rAvckIIPH/+HI6Ojh8tWywSOwMDA3h7e2PPnj1o06YNgHfJ2p49exASEqJSXqlUQqlUyuZZWFiopS1mZmYFepBqMo429UVTcdiX4h2HfSnecbSpL5qKw778H3Nz8xyVKxaJHQAMHToU3bt3R40aNVCrVi3Mnj0bL1++xDfffFPYTSMiIiJSi2KT2H399dd4+PAhxo0bh4SEBFSrVg07d+5UeaCCiIiI6FNVbBI7AAgJCcn00qsmKJVK/PjjjyqXeD/FONrUF03FYV+Kdxz2pXjH0aa+aCoO+5J3CiFy8uwsERERERV1xWKAYiIiIqLigIkdERERkZZgYkdERESkJZjYEREREWkJJnZEGsLnlIhyJy4uDqmpqYXdDCom3r59m+86YmJi8t+QfGJi9wl79OgRpk+fjv/973/w8fGBj48P/ve//2HGjBl4+PChRtpw+/ZtBAcHq6Wu169f4+DBg7h06ZLKsjdv3uC3337Ld4zLly8jPDwcV65cAQBcuXIF/fr1Q3BwMPbu3Zvv+rOjVCpx+fLlAqn75cuXCA8Pxw8//ID58+fj8ePHaqn39OnTiIuLk16vXLkSdevWhZOTE+rVq4d169apJc7AgQPxzz//qKWu7MyfPx+BgYFSu1euXAkPDw9UrFgR33//vdqSiHv37mHcuHFo3LgxKlWqhMqVK6NVq1ZYtmwZ0tLS1BIjI86qVauwfft2pKSkyJa9fPkSEyZMyHeMyMhI/Pjjj9L748CBA2jRogUaN26M8PDwfNefHXd3d8TGxhZI3Xfv3sWPP/6ILl26YPjw4dJnQkFRd4LaqlUrrFy5Eq9fv1ZrvTn1qf2j2q1bNyQlJWW5/OTJk/jss8/yHad27dqYMmUK0tPT811Xnn3012RJ7a5fvy58fX3zVcfx48eFpaWlKFWqlOjevbsYOXKkGDlypOjevbsoXbq0sLKyEidOnFBTi7MWExMjdHR08l3P1atXhbOzs1AoFEJHR0c0aNBA3L17V1qekJCQ7zg7duwQBgYGwsrKShgaGoodO3YIGxsb4efnJxo3bix0dXXFnj178tsVMWTIkEwnHR0dERgYKL3Oj0qVKonHjx8LIYS4deuWcHFxEebm5qJmzZrCyspK2Nraips3b+a7L15eXiIyMlIIIcSvv/4qjIyMxKBBg8SiRYvE4MGDhampqVi2bFm+42Ts9/Lly4tp06aJe/fu5bvOD02cOFGUKFFCtGvXTtjb24tp06YJa2trMWnSJDFlyhRhY2Mjxo0bl+84J06cEObm5sLb21vUq1dP6Orqim7duomvv/5aWFhYiM8//1wkJSXlO87x48eFhYWFMDMzE0ZGRsLNzU1cuHBBWq6O98zKlSuFnp6eqF69ujA1NRXh4eHCwsJC9OzZUwQHBwsDAwOxcePG/HZF/O9//8t00tHREX5+ftLr/DAyMhIPHjwQQghx8eJFYW5uLtzc3MRXX30lKlasKIyNjcXZs2fz3ZcdO3aIc+fOCSGESEtLExMmTBCOjo5CR0dHlCpVSkydOlWkp6fnO45CoRB6enrC3Nxc9O3bV5w8eTLfdX7ozZs3YtiwYaJ+/fpi2rRpQoh37yMTExNhYmIiOnXqJBITE9USKyYmRnTr1k24uroKQ0NDYWxsLKpUqSLGjBmjlhjVq1cXpUqVEjt37pTNT0lJEd99953Q19cXffr0yXecbdu2iVKlSonatWuLa9eu5bu+vGBiVwjUkQzVrl1b9O7dO9MPiPT0dNG7d29Rp06dfMUQQoi//vor22nWrFlqSezatGkjAgICxMOHD0VsbKwICAgQrq6u4t9//xVCqOdLysfHR/zwww9CCCHWrl0rLC0txffffy8t/+6770TTpk3zFUOIdx+41apVE40aNZJNCoVC1KxZUzRq1Cjfib1CoRD3798XQgjRpUsX8fnnn4tnz54JIYR4/vy58PPzE506dcp3X4yMjER8fLwQQojPPvtMLFmyRLZ89erVwsPDI99xFAqFiIqKEt9++60oWbKk0NfXF19++aX4+++/RVpaWr7rF0KIcuXKid9//10I8e49qKurK1atWiUt37x5s3Bzc8t3nLp164rQ0FDp9cqVK0Xt2rWFEEI8efJEVKtWTQwaNCjfcfz8/MQ333wj0tLSRFJSkujXr5+wtrYWp0+fFkKo5z1TrVo1MWfOHCGEEFFRUcLIyEiEhYVJy3/++WdRt27dfMUQ4t3+b9iwoQgKCpJNOjo6ok2bNtLr/MbIeM+0bt1atGrVSrx9+1YI8S4B69ixo/jiiy/y3Rd3d3dx4MABIYQQU6ZMEdbW1iIsLEzs2LFDzJ49W9jZ2UlJUn4oFApx8eJFMWvWLOHp6Sl0dHRE1apVxbx588STJ0/yXb8Q7/5JdXR0FMOGDROVKlUS/fv3F2XKlBGrVq0Sa9asEW5ubmLgwIH5jrNz505hZGQk2rVrJ7p27SqMjY1FSEiIGDVqlHBzcxPlypXL9z97b9++FWPHjhX6+vqid+/e4vnz5+LEiRPCw8NDlClTRuzatSvf/cjw7Nkz0b17d2FiYiLmzp2rtnpzioldAZgzZ06208iRI/P9gWtoaCguX76c5fLLly8LQ0PDfMUQ4v/OpCgUiiwndSR2tra20n+5QrxLTvv27SvKlCkjbty4oZYvKTMzMxEbGyuEePdBrqenJ30JCiHE+fPnhZ2dXb5iCCHE1KlThaurq8rZPz09PXHx4sV81y+E/EuqbNmyYvfu3bLlhw4dEk5OTvmOY21tLZ0JsLW1FTExMbLl169fF0ZGRvmO835/UlJSxPr164W/v7/Q1dUVjo6O4vvvv5f2XV4ZGRlJ/ygIIYS+vr7sDFd8fLwwNjbOV4yMODdu3JBep6WlCX19fZGQkCCEEGL37t3C0dEx33EsLS3F1atXZfOmTp0qLC0txfHjx9XynjExMZGd+dXX15ed1bp8+bKwtrbOVwwh3v2jVbp0abF8+XLZ/IJ6zzg5OUnJV4bTp08LBweHfMdRKpXScValShWxYcMG2fKtW7eq5R+I9/sjhBDHjh0TvXv3Fubm5sLIyEh06tQp31cgnJycpDP2N27cEDo6OuLPP/+Ulu/evVs4OzvnK4YQ7/6BWLRokazeihUrCiHefR40adIk34l9hhMnTojKlSsLBwcHoa+vL4KDg9V21vFDGzduFLq6usLMzExYWlrKpoLExK4AKBQK4ejoKFxcXDKdMk7L54eLi4tYsWJFlstXrFihljeco6Oj7I38oTNnzqglsStRooS4dOmSyvwBAwaI0qVLiwMHDqglsbt+/br02tTUVPYFHB8fr5ZkWIh3l8kqVKgghg0bJlJSUoQQ6v+Syris5OjoKM6fPy9brq6+dO3aVfTo0UMIIcRXX30lxowZI1s+ZcoU4enpme84H35JZfj333/Fjz/+KJydnfO9/11dXcWOHTuEEEJcu3ZN6OjoyL50t23bJlxcXPIVQwghnJ2dxcGDB6XXd+/eFQqFQrx69UoIIURcXJxa9o2lpWWmlw5nzJghLCwsxObNm/O9zSwsLMSVK1ek1x++Z27evKmWZFiId9ulbt26om3bttIZJ3W+Z3R0dKT3jLOzs8q2u3nzplr2i4ODgzhy5IgQQgg7OzvZP49CvDv21P3P0PtevnwpwsPDRb169fK9/z/2z1BcXJxa9r+hoaGIi4uTXqenpwt9fX3pdpwDBw4IGxubfMcR4t0/8NWqVRPGxsbCxMRErFy5Ui31fuj48eOiYsWKomLFimLp0qUiIiJCNhUkJnYFwMXFRaxfvz7L5epIhubPny+USqUYNGiQ+Ouvv8TRo0fF0aNHxV9//SUGDRokjIyMxIIFC/IVQwghWrVqJcaOHZvl8piYGKFQKPIdp2bNmuK3337LdNmAAQOEhYVFvreZl5eX9MUuxLs3eMalGCHefXi4urrmK8b7nj9/LgIDA4WXl5c4f/680NfXV2ti5+npKT777DNhamoqNm3aJFu+f/9+UapUqXzHuXPnjnBxcRENGjQQQ4cOFUZGRqJevXqiV69eokGDBsLAwEBs27Yt33Gy+pLKkJ6ernJWMrfGjBkjbGxsRM+ePYWrq6v47rvvRJkyZcSiRYvE4sWLhZOTU77vfRRCiG+//VZUqVJF7NixQ+zdu1f4+vqKRo0aSct37twpypUrl+849evXl53leN9PP/0klEplvt8zNWrUkP1jl5iYKLv9IzIyUlSoUCFfMd6XlpYmxo0bJ5ycnMTOnTvV/p6xsLAQlpaWQl9fX+ULfffu3WpJ7Pv37y+++OILkZqaKnr37i169uwp22YDBw4UPj4++Y7zsfeMEELljG5uubu7i3Xr1gkh3iUqBgYGsrOq69atE+XLl89XDCHe3Sbx/r1vsbGxQldXVyQnJwsh3iXd+U2G09PTxZQpU4RSqRRBQUHi6dOnYsGCBcLU1FT873//k5L+/Hr79q34/vvvhYGBgRgyZIh4/fq1WurNDSZ2BaBdu3Zi5MiRWS5XVzK0bt06Ubt2baGnpyddFtXT0xO1a9fONrHMjQMHDsiSoQ+9ePFCREdH5zvOlClTRIsWLbJc3q9fv3xvs0WLFomtW7dmuXz06NHS2Sl1Wrt2rbCzsxM6Ojpq+5IKDQ2VTR/eEDx8+HDRsWNHtcR6+vSpGDVqlPDw8BCGhobCwMBAODs7i86dO6vtAR0XFxfx6NEjtdSVlbS0NDF58mTxxRdfiClTpoj09HSxdu1a4eTkJKytrUVQUJB48eJFvuM8f/5cdOjQQXpffv7557LLmbt27VK5PJcXv/76q+jatWuWy6dNm5bvRGXz5s1i//79WS6fOnWqyllcdfjnn3+Eq6urWt8zH54xyTirlmHChAlqSeyfPXsmatSoIdzc3ES3bt2EoaGhcHZ2Fk2bNhWurq7C3NxcHD16NN9xGjVqJJ4+fZrverIza9YsYWhoKPz8/ISlpaWYO3eusLe3FyNHjhTfffedMDc3FxMmTMh3nPHjx4vSpUuLRYsWieXLl4sqVarIHpbZvHlzvu/lrVWrlnBwcBBbtmyRzb9x44aoX7++KFmypJTE5oenp6dwdXUV+/bty3ddeaUQ4hN7ZvkTcOnSJbx69Qo1atTIdPnbt29x9+5dODs7qyXe27dv8ejRIwBAyZIloa+vr5Z6SX3+++8/nDp1Cn5+fjAxMSns5pAGvHnzBqmpqTA1NS2wGEII3L59G7a2tjA0NPxkY2QW58WLF7hx4wYqVaoEAwODAolRUN6+fYtly5bh77//xs2bN5Geng4HBwfUrVsX/fr1Q+nSpdUSRxP9WbNmDY4cOYLPP/8cnTp1QnR0NMaNG4dXr16hVatWGDt2LHR08jdyWmpqKn744QesWrUKycnJ8Pf3x5w5c1CyZEkAwPHjx/HmzRs0aNAgzzE6duyIhQsXwsrKSmWZEAKzZ8/G2LFj8eLFizzHAICePXti1qxZePr0aYEfZ1lhYldANPUBQkTFV3p6OgwNDXHx4kWUL1/+k42hqTia6gsA3Lp1CzY2NjAyMiqwGJrojyYT+1u3bsHW1rZAt1lGjMz6Ehsbq5btqMnjLDMcoLiACCHg5uaG27dvF0r8GzduoHHjxoxTxGJoKo429UVTcT7Fvujo6KB8+fJqG5C6sGJoKo6m+pKeng43Nzf8999/BRpHE/3R1HeZEALly5cv0G2WsV+y6ou6kjBNHWdZxi+UqMVAYe/YFy9eYP/+/YxTxGJoKo429UVTcT7VvkybNg0jRozAhQsX1FZnYcTQVBxNxNDk539B94eJfd5o6j2TGT2NRyxGMnbsokWLUKVKFbXWPXfu3GyX37lzh3EKIYam4mhTXzQVR5v68r7AwEC8evUKVatWhYGBgcplrCdPnnwSMTQVR1N9KcjP//dpoj+a6osm4mjTfskK77ErQJaWlnj16hVSU1PVvmN1dHTg4OCQ5U3FKSkpSEhIyPfvUmpTHPaleMfRpr68b8WKFdku7969+ycRQ1NxNNWXgvz8f58m+qOpvmgijjbtl6wwsStABbljXV1d8dNPP6FDhw6ZLo+JiYG3t3e+vzy0KQ77UrzjaFNfqOgrzC92dWNi/2nhpdgCVJAHiLe3N06dOpXll4dCoYA6cnZtisO+FO842tSXD6Wnp+P69et48OAB0tPTZcvyM0SEpmNoKo4mYmgyQSjo/miqL5qIo037JSs8Y1fACmrHamqsPG2Kw74U7zja1Jf3HT16FJ07d8a///6rkjAqFAq1nBnURAxNxdFUXwDNfLFrqj9M7HNHk8fZh5jYFaCC3rGFNXDopxyHfSnecbSpLxmqVauGChUqYPz48XBwcIBCoZAtNzc3/yRiaCqOpvqiqS92TfSHiX3uaeo4ywwTuwJU0DtWmwYO1VQc9qV4x9GmvmQwMTHB2bNn4ebm9knH0FQcTfVFU1/smugPE/vc09RxlhneY1eAYmNjsWnTpgLbse+PyVOQXx7aFId9Kd5xtKkvGWrXro3r168X6BeIJmJoKo6m+lLQn/8ZNNEfTfVFE3G0ab9khYldAdLEjtWm8YU0FYd9Kd5xtKEv586dk/4eOHAghg0bhoSEBHh6eqr8VrSXl1eRjaGpOJrqy/sK8vNf0/1hYp8zhXGcZYaXYtXs/R1748YNjBkzBiNGjCiwHatN4wtpKg77UrzjaENfdHR0sn26NmNZfu4Z0kQMTcXRVF809fmvif5oqi+aiKNN+yUneMZOzapVq6ayY4ODg6W/1b1jZ8+ene86ilsc9qV4x9GGvsTFxRVY3ZqMoak4muqLpj7/NdEfTfVFE3G0ab/kBM/Yqdm///6b47LqGu6AiIqvqVOnws7OTvZFBQDLly/Hw4cPMWrUqE8ihqbiFGSMwvj8L6j+aKovmoijTfslJ5jYFSBN7VhtGl9IU3HYl+IdR5v64uLigjVr1uDzzz+XzT927Bg6duyolrMImoihqTia6oumPv810R8m9rmnqeMsU4IKjLOzszh06JDK/KNHjwoXFxe1xDhy5IhwdXUVOjo6QqFQyCYdHR21xNC2OOxL8Y6jTX0RQgilUilu3rypMv/GjRtCqVR+MjE0FUdTfdHE578QmumPpvqiiTjatF+ywnvsClBCQgIcHBxU5tvY2ODevXtqidG3b1/UqFED27Zty3RMHnXRpjjsS/GOo019AQAnJyccOnQIrq6usvmHDh2Co6PjJxNDU3E01RdNfP4DmumPpvqiiTjatF+yVKBpYzHn5uYmVq5cqTL/t99+E66urmqJYWxsLGJjY9VSV3GJw74U7zja1BchhPjpp5+EtbW1WL58uYiPjxfx8fFi2bJlwtraWkyZMuWTiaGpOJrqiyY+/4XQTH801RdNxNGm/ZIVJnYFSBM71tfXV+zYsUMtdRWXOOxL8Y6jTX0RQoj09HQxcuRIYWhoKHR0dISOjo4wNjYW48eP/6RiaCqOpvqiqS92TfSHiX3uaeo4ywwfnihAQgh89913mDt3LlJSUgAAhoaGGDVqFMaNG5fnerVpfCFNxWFfinccbepLVl68eIHLly/DyMgI5cuXh1KpVGv9moqhqTgFHaOgPv+zUpD90VRfNBFHm/ZLVpjYaYC6d6w2DRyqqTjsS/GOo019oU9LYXyxFxQm9p8GJnafIG0aX0hTcdiX4h1Hm/pCRJQdJnafOG0aX0hTcdiX4h1Hm/pCRKSiwO/iowKlTeMLaSoO+1K842hTX4iIPqRT2Ikl5Y82jS+kqTjsS/GOo019ISL6EBO7T1zGIIgfKqiBQ7UhDvtSvONoU1+IiD7EX574xPXq1QuDBw/G27dv0bhxYwDAnj17MHLkSAwbNoxxCimGpuJoU180FUeb+kJEpKKwrwVT/mjTwKGaisO+FO842tQXIqIP8alYLaFN4wtpKg77UrzjaFNfiIgyMLEjIiIi0hJ8eIKIiIhISzCxIyIiItISTOyIiIiItAQTOyIiIiItwcSOiIiISEswsSMi+v8SEhIwcOBAlC1bFkqlEk5OTmjVqhX27NkDAHBxccHs2bNV1gsNDUW1atVkrxUKBRQKBXR1deHk5ITevXvjyZMnKusePnwYLVu2hKWlJQwNDeHp6YmwsDCkpaXJymXUd/ToUdn85ORkWFtbQ6FQIDo6Ot/bgIg+bUzsiIgAxMfHw9vbG3v37sWMGTNw/vx57Ny5E76+vhgwYECu66tcuTLu3buHW7duITw8HDt37kS/fv1kZf744w80bNgQpUuXxr59+3DlyhV8++23mDRpEjp27IgPR6NycnJCeHi4Sh2mpqa57zARaSX+pBgREYD+/ftDoVDg+PHjMDExkeZXrlwZwcHBua5PT08P9vb2AIBSpUrhq6++kiVlL1++RK9evfDll19iyZIl0vyePXvCzs4OX375JTZs2ICvv/5aWta9e3fMnTsXs2fPhpGREQBg+fLl6N69OyZOnJjrNhKR9uEZOyIq9p48eYKdO3diwIABsqQug4WFRb7qj4+Px65du2BgYCDN2717Nx4/fozhw4erlG/VqhUqVKiAtWvXyuZ7e3vDxcUFv//+OwDg1q1bOHDgALp165av9hGR9mBiR0TF3vXr1yGEQMWKFT9adtSoUTA1NZVNU6ZMUSl3/vx5mJqawsjICK6urrh48SJGjRolLb927RoAoFKlSpnGqVixolTmfcHBwVi+fDkAICIiAi1btoSNjU2O+klE2o+XYomo2MvNLyuOGDECQUFBsnlz587FgQMHZPPc3d2xZcsWvHnzBqtWrUJMTAwGDhyYr9gA0LVrV3z33Xe4efMmIiIiMHfu3FytT0TajWfsiKjYK1++PBQKBa5cufLRsiVLloSbm5tssrKyUilnYGAANzc3VKlSBdOmTYOuri7Gjx8vLa9QoQIA4PLly5nGuXz5slTmfdbW1vjiiy/Qo0cPvHnzBi1atMhpN4moGGBiR0TFnpWVFfz9/bFgwQK8fPlSZfmzZ8/yHWPMmDH4+eefcffuXQBAs2bNYGVlhZkzZ6qU3bJlC2JjY9GpU6dM6woODkZ0dDQCAwOhq6ub77YRkfZgYkdEBGDBggVIS0tDrVq18PvvvyM2NhaXL1/G3Llz4ePjk+/6fXx84OXlJd2PZ2Jigl9++QV//fUXevfujXPnziE+Ph7Lli1DUFAQ2rdvjw4dOmRaV/PmzfHw4UNMmDAh3+0iIu3CxI6ICEDZsmVx+vRp+Pr6YtiwYahSpQqaNm2KPXv2YNGiRWqJMWTIECxduhS3b98GALRv3x779u3DrVu3UL9+fbi7u2PWrFn44YcfsG7dOigUikzrUSgUKFmypOwpWyIiAFCI3N65S0RERERFEs/YEREREWkJJnZEREREWoKJHREREZGWYGJHREREpCWY2BERERFpCSZ2RERERFqCiR0RERGRlmBiR0RERKQlmNgRERERaQkmdkRERERagokdERERkZZgYkdERESkJf4fsVt6lfqKwicAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    }
  ]
}